
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>input_tempfile - 11 region(s) - antiSMASH results</title>
  <style>img {width: 20px; height: 20px;}</style>
  <link rel="stylesheet" type="text/css" href="css/bacteria.css">
  <meta property="og:title" content="input_tempfile - 11 record(s) - 11 region(s)" />
  <meta property="og:description" content="">
  <meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
  <meta property="og:image:alt" content="antiSMASH logo" />
  <meta property="og:image:width" content="600" />
  <meta property="og:image:height" content="600" />
  <meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/input_tempfile/index.html" />
  <meta property="og:type" content="website" />
  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
  <div id="header">
      <nav>
    <div class="main-link">
     <div>
        <a href="https://antismash.secondarymetabolites.org/">
          <img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
        </a>
       </div>
       <div>
        <a href="https://antismash.secondarymetabolites.org/">
          antiSMASH version 6.1.1
        </a>
      </div>
    </div>
    <div class="custom-description"></div>
    <div class="ancillary-links">
      <div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
        <ul class="dropdown-options">
        
        <li><a href="input_tempfile.zip">Download all results</a></li>
        
          <li><a href="input_tempfile.gbk">Download GenBank summary file</a></li>
        
        </ul>
      </div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
      <div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
      <div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
    </div>
  </nav>
    <div id="buttons">
      <span id="region-type">Select genomic region:</span>
      <div id="regionbuttons">
        <div class="arrow-left" id="prev-region"></div>
        <div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
        <div style="display: flex; flex-wrap: wrap">
        
          
            <div class="regbutton NRPS NRPS-like r6c1">
              <a href="#r6c1">6.1</a>
            </div>
          
        
          
            <div class="regbutton terpene terpene r25c1">
              <a href="#r25c1">25.1</a>
            </div>
          
        
          
            <div class="regbutton RiPP r44c1">
              <a href="#r44c1">44.1</a>
            </div>
          
        
          
            <div class="regbutton NRPS NRPS r49c1">
              <a href="#r49c1">49.1</a>
            </div>
          
        
          
            <div class="regbutton NRPS NRPS r74c1">
              <a href="#r74c1">74.1</a>
            </div>
          
        
          
            <div class="regbutton other siderophore r93c1">
              <a href="#r93c1">93.1</a>
            </div>
          
        
          
            <div class="regbutton other betalactone r138c1">
              <a href="#r138c1">138.1</a>
            </div>
          
        
          
            <div class="regbutton NRPS NRPS r199c1">
              <a href="#r199c1">199.1</a>
            </div>
          
        
          
            <div class="regbutton NRPS NRPS r310c1">
              <a href="#r310c1">310.1</a>
            </div>
          
        
          
            <div class="regbutton RiPP RiPP-like r462c1">
              <a href="#r462c1">462.1</a>
            </div>
          
        
          
            <div class="regbutton RiPP RiPP-like r532c1">
              <a href="#r532c1">532.1</a>
            </div>
          
        
        </div>
        <div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
      </div>
    </div>
  </div>

  <!-- overview page -->
  
  <div class="page" id="overview">
   
   
     <h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>
   
   <div class="overview-layout">
    <div id="record-tables">
     <div id="single-record-tables">
     <div class="record-overview-header">
        <strong>c00006_NODE_6_..</strong> (original name was: NODE_6_length_75838_cov_28.2576)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="c00006_NODE_6_.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r6c1">
     <td class="regbutton NRPS NRPS-like r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">3,118</td>
     <td class="digits table-split-left">46,699</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00025_NODE_25..</strong> (original name was: NODE_25_length_41388_cov_21.564)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-25" data-filename="c00025_NODE_25.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-25" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r25c1">
     <td class="regbutton terpene terpene r25c1">
      <a href="#r25c1">Region&nbsp25.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">9,528</td>
     <td class="digits table-split-left">31,381</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000916/1" target="_blank">molybdenum cofactor</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00044_NODE_44..</strong> (original name was: NODE_44_length_27858_cov_28.2843)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-44" data-filename="c00044_NODE_44.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-44" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r44c1">
     <td class="regbutton RiPP r44c1">
      <a href="#r44c1">Region&nbsp44.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">20,062</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00049_NODE_49..</strong> (original name was: NODE_49_length_25382_cov_24.5764)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-49" data-filename="c00049_NODE_49.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-49" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r49c1">
     <td class="regbutton NRPS NRPS r49c1">
      <a href="#r49c1">Region&nbsp49.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">25,382</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000397/1" target="_blank">nostocyclopeptide A2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00074_NODE_74..</strong> (original name was: NODE_74_length_18380_cov_17.0523)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-74" data-filename="c00074_NODE_74.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-74" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r74c1">
     <td class="regbutton NRPS NRPS r74c1">
      <a href="#r74c1">Region&nbsp74.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">18,380</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000309/1" target="_blank">bacillibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 46%, #ffffff00 46%)">46%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00093_NODE_93..</strong> (original name was: NODE_93_length_13696_cov_12.5188)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-93" data-filename="c00093_NODE_93.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-93" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r93c1">
     <td class="regbutton other siderophore r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">12,527</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000942/1" target="_blank">petrobactin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00138_NODE_13..</strong> (original name was: NODE_138_length_9692_cov_17.8376)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-138" data-filename="c00138_NODE_13.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-138" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r138c1">
     <td class="regbutton other betalactone r138c1">
      <a href="#r138c1">Region&nbsp138.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,692</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001095/1" target="_blank">fengycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00199_NODE_19..</strong> (original name was: NODE_199_length_6251_cov_9.32707)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-199" data-filename="c00199_NODE_19.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-199" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r199c1">
     <td class="regbutton NRPS NRPS r199c1">
      <a href="#r199c1">Region&nbsp199.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,251</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00310_NODE_31..</strong> (original name was: NODE_310_length_3541_cov_6.02343)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-310" data-filename="c00310_NODE_31.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-310" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r310c1">
     <td class="regbutton NRPS NRPS r310c1">
      <a href="#r310c1">Region&nbsp310.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,541</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00462_NODE_46..</strong> (original name was: NODE_462_length_1902_cov_6.66817)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-462" data-filename="c00462_NODE_46.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-462" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r462c1">
     <td class="regbutton RiPP RiPP-like r462c1">
      <a href="#r462c1">Region&nbsp462.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,902</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div><div class="record-overview-header">
        <strong>c00532_NODE_53..</strong> (original name was: NODE_532_length_1409_cov_3.08346)
        <div class="download-container">
          <div class="download-icon-inverse download-svg" data-tag="record-minimap-532" data-filename="c00532_NODE_53.._overview.svg">
          </div>
        </div>
      </div>
      <div class="record-overview-details">
        <svg id="record-minimap-532" class="record-overview-svg" height="1" width="1"></svg>
        <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 <tr class="linked-row odd" data-anchor="#r532c1">
     <td class="regbutton RiPP RiPP-like r532c1">
      <a href="#r532c1">Region&nbsp532.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,409</td>
     
      <td colspan="3"></td>
     
   </tr>
 
 </tbody>
</table>
      </div>
     </div>
     
     <div id="compact-record-table" style="display: none;">
      <div class="record-table">
       <table class="region-table">
 <thead>
  <tr>
   <th>Region</th>
   <th>Type</th>
   <th>From</th>
   <th>To</th>
   <th colspan="2">Most similar known cluster</th>
   <th>Similarity</th>
  </tr>
 </thead>
 <tbody>
 
 
  <tr class="linked-row odd" data-anchor="#r6c1">
     <td class="regbutton NRPS NRPS-like r6c1">
      <a href="#r6c1">Region&nbsp6.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
     </td>
     <td class="digits">3,118</td>
     <td class="digits table-split-left">46,699</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r25c1">
     <td class="regbutton terpene terpene r25c1">
      <a href="#r25c1">Region&nbsp25.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
     </td>
     <td class="digits">9,528</td>
     <td class="digits table-split-left">31,381</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000916/1" target="_blank">molybdenum cofactor</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 17%, #ffffff00 17%)">17%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r44c1">
     <td class="regbutton RiPP r44c1">
      <a href="#r44c1">Region&nbsp44.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">20,062</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r49c1">
     <td class="regbutton NRPS NRPS r49c1">
      <a href="#r49c1">Region&nbsp49.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">25,382</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000397/1" target="_blank">nostocyclopeptide A2</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 28%, #ffffff00 28%)">28%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r74c1">
     <td class="regbutton NRPS NRPS r74c1">
      <a href="#r74c1">Region&nbsp74.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">18,380</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000309/1" target="_blank">bacillibactin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 46%, #ffffff00 46%)">46%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r93c1">
     <td class="regbutton other siderophore r93c1">
      <a href="#r93c1">Region&nbsp93.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#siderophore" target="_blank">siderophore</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">12,527</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000942/1" target="_blank">petrobactin</a></td>
      <td>Other</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r138c1">
     <td class="regbutton other betalactone r138c1">
      <a href="#r138c1">Region&nbsp138.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#betalactone" target="_blank">betalactone</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">9,692</td>
     
      
        
      
      <td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001095/1" target="_blank">fengycin</a></td>
      <td>NRP</td>
      <td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r199c1">
     <td class="regbutton NRPS NRPS r199c1">
      <a href="#r199c1">Region&nbsp199.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">6,251</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r310c1">
     <td class="regbutton NRPS NRPS r310c1">
      <a href="#r310c1">Region&nbsp310.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">3,541</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row even" data-anchor="#r462c1">
     <td class="regbutton RiPP RiPP-like r462c1">
      <a href="#r462c1">Region&nbsp462.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,902</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
  <tr class="linked-row odd" data-anchor="#r532c1">
     <td class="regbutton RiPP RiPP-like r532c1">
      <a href="#r532c1">Region&nbsp532.1</a>
     </td>
     <td>
       
       <a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
     </td>
     <td class="digits">1</td>
     <td class="digits table-split-left">1,409</td>
     
      <td colspan="3"></td>
     
   </tr>
  
 
 </tbody>
</table>
      </div>
     </div>
     
     
     <dl id="regionless-records">
      <dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>
       
         <dd><strong>c00001_NODE_1_..</strong> (original name was: NODE_1_length_145060_cov_26.2394)</dd>
       
         <dd><strong>c00002_NODE_2_..</strong> (original name was: NODE_2_length_123507_cov_39.0785)</dd>
       
         <dd><strong>c00003_NODE_3_..</strong> (original name was: NODE_3_length_118863_cov_33.7155)</dd>
       
         <dd><strong>c00004_NODE_4_..</strong> (original name was: NODE_4_length_111815_cov_41.1102)</dd>
       
         <dd><strong>c00005_NODE_5_..</strong> (original name was: NODE_5_length_79492_cov_34.127)</dd>
       
         <dd><strong>c00007_NODE_7_..</strong> (original name was: NODE_7_length_75071_cov_20.1658)</dd>
       
         <dd><strong>c00008_NODE_8_..</strong> (original name was: NODE_8_length_72048_cov_21.6786)</dd>
       
         <dd><strong>c00009_NODE_9_..</strong> (original name was: NODE_9_length_61892_cov_25.068)</dd>
       
         <dd><strong>c00010_NODE_10..</strong> (original name was: NODE_10_length_59634_cov_34.9257)</dd>
       
         <dd><strong>c00011_NODE_11..</strong> (original name was: NODE_11_length_59147_cov_66.3088)</dd>
       
         <dd><strong>c00012_NODE_12..</strong> (original name was: NODE_12_length_58766_cov_30.4123)</dd>
       
         <dd><strong>c00013_NODE_13..</strong> (original name was: NODE_13_length_58275_cov_34.9688)</dd>
       
         <dd><strong>c00014_NODE_14..</strong> (original name was: NODE_14_length_56339_cov_27.4719)</dd>
       
         <dd><strong>c00015_NODE_15..</strong> (original name was: NODE_15_length_51334_cov_25.1588)</dd>
       
         <dd><strong>c00016_NODE_16..</strong> (original name was: NODE_16_length_50913_cov_29.7127)</dd>
       
         <dd><strong>c00017_NODE_17..</strong> (original name was: NODE_17_length_50276_cov_34.1585)</dd>
       
         <dd><strong>c00018_NODE_18..</strong> (original name was: NODE_18_length_50218_cov_34.4958)</dd>
       
         <dd><strong>c00019_NODE_19..</strong> (original name was: NODE_19_length_49570_cov_31.5516)</dd>
       
         <dd><strong>c00020_NODE_20..</strong> (original name was: NODE_20_length_48595_cov_40.6964)</dd>
       
         <dd><strong>c00021_NODE_21..</strong> (original name was: NODE_21_length_46572_cov_58.7618)</dd>
       
         <dd><strong>c00022_NODE_22..</strong> (original name was: NODE_22_length_46186_cov_28.1662)</dd>
       
         <dd><strong>c00023_NODE_23..</strong> (original name was: NODE_23_length_42682_cov_37.7773)</dd>
       
         <dd><strong>c00024_NODE_24..</strong> (original name was: NODE_24_length_42538_cov_28.8514)</dd>
       
         <dd><strong>c00026_NODE_26..</strong> (original name was: NODE_26_length_40612_cov_19.392)</dd>
       
         <dd><strong>c00027_NODE_27..</strong> (original name was: NODE_27_length_40194_cov_26.2782)</dd>
       
         <dd><strong>c00028_NODE_28..</strong> (original name was: NODE_28_length_39913_cov_23.5204)</dd>
       
         <dd><strong>c00029_NODE_29..</strong> (original name was: NODE_29_length_39831_cov_22.4167)</dd>
       
         <dd><strong>c00030_NODE_30..</strong> (original name was: NODE_30_length_39823_cov_42.2894)</dd>
       
         <dd><strong>c00031_NODE_31..</strong> (original name was: NODE_31_length_39589_cov_39.6587)</dd>
       
         <dd><strong>c00032_NODE_32..</strong> (original name was: NODE_32_length_39311_cov_26.6906)</dd>
       
         <dd><strong>c00033_NODE_33..</strong> (original name was: NODE_33_length_37089_cov_18.9543)</dd>
       
         <dd><strong>c00034_NODE_34..</strong> (original name was: NODE_34_length_36771_cov_28.4555)</dd>
       
         <dd><strong>c00035_NODE_35..</strong> (original name was: NODE_35_length_36222_cov_32.4947)</dd>
       
         <dd><strong>c00036_NODE_36..</strong> (original name was: NODE_36_length_35247_cov_24.0317)</dd>
       
         <dd><strong>c00037_NODE_37..</strong> (original name was: NODE_37_length_33651_cov_16.9873)</dd>
       
         <dd><strong>c00038_NODE_38..</strong> (original name was: NODE_38_length_31147_cov_26.0128)</dd>
       
         <dd><strong>c00039_NODE_39..</strong> (original name was: NODE_39_length_31144_cov_30.2842)</dd>
       
         <dd><strong>c00040_NODE_40..</strong> (original name was: NODE_40_length_30992_cov_23.1442)</dd>
       
         <dd><strong>c00041_NODE_41..</strong> (original name was: NODE_41_length_29921_cov_11.4197)</dd>
       
         <dd><strong>c00042_NODE_42..</strong> (original name was: NODE_42_length_29821_cov_31.9082)</dd>
       
         <dd><strong>c00043_NODE_43..</strong> (original name was: NODE_43_length_28297_cov_15.2738)</dd>
       
         <dd><strong>c00045_NODE_45..</strong> (original name was: NODE_45_length_26725_cov_28.706)</dd>
       
         <dd><strong>c00046_NODE_46..</strong> (original name was: NODE_46_length_26523_cov_26.7661)</dd>
       
         <dd><strong>c00047_NODE_47..</strong> (original name was: NODE_47_length_25950_cov_25.1846)</dd>
       
         <dd><strong>c00048_NODE_48..</strong> (original name was: NODE_48_length_25474_cov_12.5044)</dd>
       
         <dd><strong>c00050_NODE_50..</strong> (original name was: NODE_50_length_24833_cov_25.1809)</dd>
       
         <dd><strong>c00051_NODE_51..</strong> (original name was: NODE_51_length_24823_cov_21.7703)</dd>
       
         <dd><strong>c00052_NODE_52..</strong> (original name was: NODE_52_length_24553_cov_27.7978)</dd>
       
         <dd><strong>c00053_NODE_53..</strong> (original name was: NODE_53_length_24378_cov_37.0564)</dd>
       
         <dd><strong>c00054_NODE_54..</strong> (original name was: NODE_54_length_24104_cov_21.0043)</dd>
       
         <dd><strong>c00055_NODE_55..</strong> (original name was: NODE_55_length_24026_cov_29.0248)</dd>
       
         <dd><strong>c00056_NODE_56..</strong> (original name was: NODE_56_length_23843_cov_15.5214)</dd>
       
         <dd><strong>c00057_NODE_57..</strong> (original name was: NODE_57_length_23790_cov_24.9713)</dd>
       
         <dd><strong>c00058_NODE_58..</strong> (original name was: NODE_58_length_23754_cov_16.7191)</dd>
       
         <dd><strong>c00059_NODE_59..</strong> (original name was: NODE_59_length_23269_cov_38.4551)</dd>
       
         <dd><strong>c00060_NODE_60..</strong> (original name was: NODE_60_length_23245_cov_21.0793)</dd>
       
         <dd><strong>c00061_NODE_61..</strong> (original name was: NODE_61_length_23195_cov_23.2785)</dd>
       
         <dd><strong>c00062_NODE_62..</strong> (original name was: NODE_62_length_22830_cov_15.0153)</dd>
       
         <dd><strong>c00063_NODE_63..</strong> (original name was: NODE_63_length_22415_cov_16.1707)</dd>
       
         <dd><strong>c00064_NODE_64..</strong> (original name was: NODE_64_length_21694_cov_16.1939)</dd>
       
         <dd><strong>c00065_NODE_65..</strong> (original name was: NODE_65_length_21594_cov_36.3577)</dd>
       
         <dd><strong>c00066_NODE_66..</strong> (original name was: NODE_66_length_20703_cov_17.0931)</dd>
       
         <dd><strong>c00067_NODE_67..</strong> (original name was: NODE_67_length_20336_cov_23.2854)</dd>
       
         <dd><strong>c00068_NODE_68..</strong> (original name was: NODE_68_length_19920_cov_33.7766)</dd>
       
         <dd><strong>c00069_NODE_69..</strong> (original name was: NODE_69_length_19758_cov_22.9743)</dd>
       
         <dd><strong>c00070_NODE_70..</strong> (original name was: NODE_70_length_19291_cov_13.3653)</dd>
       
         <dd><strong>c00071_NODE_71..</strong> (original name was: NODE_71_length_19172_cov_17.1511)</dd>
       
         <dd><strong>c00072_NODE_72..</strong> (original name was: NODE_72_length_18749_cov_9.5159)</dd>
       
         <dd><strong>c00073_NODE_73..</strong> (original name was: NODE_73_length_18427_cov_49.0618)</dd>
       
         <dd><strong>c00075_NODE_75..</strong> (original name was: NODE_75_length_18290_cov_37.3575)</dd>
       
         <dd><strong>c00076_NODE_76..</strong> (original name was: NODE_76_length_17426_cov_24.74)</dd>
       
         <dd><strong>c00077_NODE_77..</strong> (original name was: NODE_77_length_17403_cov_31.2558)</dd>
       
         <dd><strong>c00078_NODE_78..</strong> (original name was: NODE_78_length_17349_cov_14.3926)</dd>
       
         <dd><strong>c00079_NODE_79..</strong> (original name was: NODE_79_length_16837_cov_14.0254)</dd>
       
         <dd><strong>c00080_NODE_80..</strong> (original name was: NODE_80_length_16558_cov_11.2618)</dd>
       
         <dd><strong>c00081_NODE_81..</strong> (original name was: NODE_81_length_16499_cov_51.6756)</dd>
       
         <dd><strong>c00082_NODE_82..</strong> (original name was: NODE_82_length_16480_cov_17.1091)</dd>
       
         <dd><strong>c00083_NODE_83..</strong> (original name was: NODE_83_length_16103_cov_23.7164)</dd>
       
         <dd><strong>c00084_NODE_84..</strong> (original name was: NODE_84_length_15868_cov_20.8756)</dd>
       
         <dd><strong>c00085_NODE_85..</strong> (original name was: NODE_85_length_15291_cov_56.9214)</dd>
       
         <dd><strong>c00086_NODE_86..</strong> (original name was: NODE_86_length_15138_cov_16.9378)</dd>
       
         <dd><strong>c00087_NODE_87..</strong> (original name was: NODE_87_length_15001_cov_12.5917)</dd>
       
         <dd><strong>c00088_NODE_88..</strong> (original name was: NODE_88_length_14918_cov_15.9004)</dd>
       
         <dd><strong>c00089_NODE_89..</strong> (original name was: NODE_89_length_14031_cov_22.0367)</dd>
       
         <dd><strong>c00090_NODE_90..</strong> (original name was: NODE_90_length_13924_cov_20.5343)</dd>
       
         <dd><strong>c00091_NODE_91..</strong> (original name was: NODE_91_length_13884_cov_13.0354)</dd>
       
         <dd><strong>c00092_NODE_92..</strong> (original name was: NODE_92_length_13883_cov_14.9018)</dd>
       
         <dd><strong>c00094_NODE_94..</strong> (original name was: NODE_94_length_13665_cov_22.3223)</dd>
       
         <dd><strong>c00095_NODE_95..</strong> (original name was: NODE_95_length_13612_cov_21.627)</dd>
       
         <dd><strong>c00096_NODE_96..</strong> (original name was: NODE_96_length_13574_cov_13.9342)</dd>
       
         <dd><strong>c00097_NODE_97..</strong> (original name was: NODE_97_length_13546_cov_26.7365)</dd>
       
         <dd><strong>c00098_NODE_98..</strong> (original name was: NODE_98_length_13502_cov_14.6292)</dd>
       
         <dd><strong>c00099_NODE_99..</strong> (original name was: NODE_99_length_13247_cov_28.5489)</dd>
       
         <dd><strong>c00100_NODE_10..</strong> (original name was: NODE_100_length_13216_cov_10.9448)</dd>
       
         <dd><strong>c00101_NODE_10..</strong> (original name was: NODE_101_length_12923_cov_17.7123)</dd>
       
         <dd><strong>c00102_NODE_10..</strong> (original name was: NODE_102_length_12805_cov_20.5954)</dd>
       
         <dd><strong>c00103_NODE_10..</strong> (original name was: NODE_103_length_12601_cov_40.0836)</dd>
       
         <dd><strong>c00104_NODE_10..</strong> (original name was: NODE_104_length_12600_cov_19.2684)</dd>
       
         <dd><strong>c00105_NODE_10..</strong> (original name was: NODE_105_length_12517_cov_15.8584)</dd>
       
         <dd><strong>c00106_NODE_10..</strong> (original name was: NODE_106_length_12317_cov_14.5057)</dd>
       
         <dd><strong>c00107_NODE_10..</strong> (original name was: NODE_107_length_12275_cov_14.9658)</dd>
       
         <dd><strong>c00108_NODE_10..</strong> (original name was: NODE_108_length_12146_cov_11.4264)</dd>
       
         <dd><strong>c00109_NODE_10..</strong> (original name was: NODE_109_length_12143_cov_29.3946)</dd>
       
         <dd><strong>c00110_NODE_11..</strong> (original name was: NODE_110_length_12074_cov_25.0259)</dd>
       
         <dd><strong>c00111_NODE_11..</strong> (original name was: NODE_111_length_12006_cov_12.2889)</dd>
       
         <dd><strong>c00112_NODE_11..</strong> (original name was: NODE_112_length_11818_cov_13.1188)</dd>
       
         <dd><strong>c00113_NODE_11..</strong> (original name was: NODE_113_length_11673_cov_21.1499)</dd>
       
         <dd><strong>c00114_NODE_11..</strong> (original name was: NODE_114_length_11596_cov_25.5551)</dd>
       
         <dd><strong>c00115_NODE_11..</strong> (original name was: NODE_115_length_11541_cov_13.0127)</dd>
       
         <dd><strong>c00116_NODE_11..</strong> (original name was: NODE_116_length_11436_cov_11.6901)</dd>
       
         <dd><strong>c00117_NODE_11..</strong> (original name was: NODE_117_length_11432_cov_14.9334)</dd>
       
         <dd><strong>c00118_NODE_11..</strong> (original name was: NODE_118_length_11191_cov_13.5839)</dd>
       
         <dd><strong>c00119_NODE_11..</strong> (original name was: NODE_119_length_11187_cov_18.0029)</dd>
       
         <dd><strong>c00120_NODE_12..</strong> (original name was: NODE_120_length_11160_cov_37.7918)</dd>
       
         <dd><strong>c00121_NODE_12..</strong> (original name was: NODE_121_length_11138_cov_13.4442)</dd>
       
         <dd><strong>c00122_NODE_12..</strong> (original name was: NODE_122_length_11075_cov_15.8663)</dd>
       
         <dd><strong>c00123_NODE_12..</strong> (original name was: NODE_123_length_10956_cov_12.5128)</dd>
       
         <dd><strong>c00124_NODE_12..</strong> (original name was: NODE_124_length_10933_cov_14.8261)</dd>
       
         <dd><strong>c00125_NODE_12..</strong> (original name was: NODE_125_length_10697_cov_17.4563)</dd>
       
         <dd><strong>c00126_NODE_12..</strong> (original name was: NODE_126_length_10611_cov_13.1662)</dd>
       
         <dd><strong>c00127_NODE_12..</strong> (original name was: NODE_127_length_10603_cov_15.7931)</dd>
       
         <dd><strong>c00128_NODE_12..</strong> (original name was: NODE_128_length_10481_cov_13.8958)</dd>
       
         <dd><strong>c00129_NODE_12..</strong> (original name was: NODE_129_length_10345_cov_13.1837)</dd>
       
         <dd><strong>c00130_NODE_13..</strong> (original name was: NODE_130_length_10270_cov_16.1074)</dd>
       
         <dd><strong>c00131_NODE_13..</strong> (original name was: NODE_131_length_10225_cov_12.4439)</dd>
       
         <dd><strong>c00132_NODE_13..</strong> (original name was: NODE_132_length_10185_cov_14.2053)</dd>
       
         <dd><strong>c00133_NODE_13..</strong> (original name was: NODE_133_length_10168_cov_16.9475)</dd>
       
         <dd><strong>c00134_NODE_13..</strong> (original name was: NODE_134_length_10132_cov_12.5936)</dd>
       
         <dd><strong>c00135_NODE_13..</strong> (original name was: NODE_135_length_9964_cov_28.9525)</dd>
       
         <dd><strong>c00136_NODE_13..</strong> (original name was: NODE_136_length_9958_cov_22.1933)</dd>
       
         <dd><strong>c00137_NODE_13..</strong> (original name was: NODE_137_length_9717_cov_9.32565)</dd>
       
         <dd><strong>c00139_NODE_13..</strong> (original name was: NODE_139_length_9620_cov_13.8638)</dd>
       
         <dd><strong>c00140_NODE_14..</strong> (original name was: NODE_140_length_9599_cov_9.27692)</dd>
       
         <dd><strong>c00141_NODE_14..</strong> (original name was: NODE_141_length_9598_cov_15.0532)</dd>
       
         <dd><strong>c00142_NODE_14..</strong> (original name was: NODE_142_length_9584_cov_26.2538)</dd>
       
         <dd><strong>c00143_NODE_14..</strong> (original name was: NODE_143_length_9571_cov_26.0274)</dd>
       
         <dd><strong>c00144_NODE_14..</strong> (original name was: NODE_144_length_9493_cov_13.3227)</dd>
       
         <dd><strong>c00145_NODE_14..</strong> (original name was: NODE_145_length_9410_cov_10.7687)</dd>
       
         <dd><strong>c00146_NODE_14..</strong> (original name was: NODE_146_length_9374_cov_22.8593)</dd>
       
         <dd><strong>c00147_NODE_14..</strong> (original name was: NODE_147_length_9356_cov_14.7989)</dd>
       
         <dd><strong>c00148_NODE_14..</strong> (original name was: NODE_148_length_9232_cov_18.6214)</dd>
       
         <dd><strong>c00149_NODE_14..</strong> (original name was: NODE_149_length_9199_cov_12.6113)</dd>
       
         <dd><strong>c00150_NODE_15..</strong> (original name was: NODE_150_length_9127_cov_12.3752)</dd>
       
         <dd><strong>c00151_NODE_15..</strong> (original name was: NODE_151_length_9105_cov_12.3819)</dd>
       
         <dd><strong>c00152_NODE_15..</strong> (original name was: NODE_152_length_8968_cov_23.8013)</dd>
       
         <dd><strong>c00153_NODE_15..</strong> (original name was: NODE_153_length_8927_cov_77.2975)</dd>
       
         <dd><strong>c00154_NODE_15..</strong> (original name was: NODE_154_length_8848_cov_23.7529)</dd>
       
         <dd><strong>c00155_NODE_15..</strong> (original name was: NODE_155_length_8695_cov_11.1194)</dd>
       
         <dd><strong>c00156_NODE_15..</strong> (original name was: NODE_156_length_8672_cov_8.91925)</dd>
       
         <dd><strong>c00157_NODE_15..</strong> (original name was: NODE_157_length_8509_cov_10.2005)</dd>
       
         <dd><strong>c00158_NODE_15..</strong> (original name was: NODE_158_length_8487_cov_10.9234)</dd>
       
         <dd><strong>c00159_NODE_15..</strong> (original name was: NODE_159_length_8413_cov_13.7305)</dd>
       
         <dd><strong>c00160_NODE_16..</strong> (original name was: NODE_160_length_8302_cov_21.4838)</dd>
       
         <dd><strong>c00161_NODE_16..</strong> (original name was: NODE_161_length_8247_cov_9.85148)</dd>
       
         <dd><strong>c00162_NODE_16..</strong> (original name was: NODE_162_length_8240_cov_14.7124)</dd>
       
         <dd><strong>c00163_NODE_16..</strong> (original name was: NODE_163_length_8094_cov_9.83959)</dd>
       
         <dd><strong>c00164_NODE_16..</strong> (original name was: NODE_164_length_8066_cov_13.3558)</dd>
       
         <dd><strong>c00165_NODE_16..</strong> (original name was: NODE_165_length_8055_cov_9.46632)</dd>
       
         <dd><strong>c00166_NODE_16..</strong> (original name was: NODE_166_length_8032_cov_11.5394)</dd>
       
         <dd><strong>c00167_NODE_16..</strong> (original name was: NODE_167_length_8016_cov_33.9198)</dd>
       
         <dd><strong>c00168_NODE_16..</strong> (original name was: NODE_168_length_8012_cov_23.2526)</dd>
       
         <dd><strong>c00169_NODE_16..</strong> (original name was: NODE_169_length_8004_cov_13.4358)</dd>
       
         <dd><strong>c00170_NODE_17..</strong> (original name was: NODE_170_length_7922_cov_10.5502)</dd>
       
         <dd><strong>c00171_NODE_17..</strong> (original name was: NODE_171_length_7861_cov_21.1682)</dd>
       
         <dd><strong>c00172_NODE_17..</strong> (original name was: NODE_172_length_7790_cov_9.86141)</dd>
       
         <dd><strong>c00173_NODE_17..</strong> (original name was: NODE_173_length_7732_cov_9.30638)</dd>
       
         <dd><strong>c00174_NODE_17..</strong> (original name was: NODE_174_length_7692_cov_13.8131)</dd>
       
         <dd><strong>c00175_NODE_17..</strong> (original name was: NODE_175_length_7686_cov_9.33483)</dd>
       
         <dd><strong>c00176_NODE_17..</strong> (original name was: NODE_176_length_7685_cov_34.9173)</dd>
       
         <dd><strong>c00177_NODE_17..</strong> (original name was: NODE_177_length_7527_cov_10.0954)</dd>
       
         <dd><strong>c00178_NODE_17..</strong> (original name was: NODE_178_length_7443_cov_10.9148)</dd>
       
         <dd><strong>c00179_NODE_17..</strong> (original name was: NODE_179_length_7436_cov_12.0718)</dd>
       
         <dd><strong>c00180_NODE_18..</strong> (original name was: NODE_180_length_7433_cov_27.9797)</dd>
       
         <dd><strong>c00181_NODE_18..</strong> (original name was: NODE_181_length_7425_cov_15.697)</dd>
       
         <dd><strong>c00182_NODE_18..</strong> (original name was: NODE_182_length_7376_cov_47.5975)</dd>
       
         <dd><strong>c00183_NODE_18..</strong> (original name was: NODE_183_length_7345_cov_15.7006)</dd>
       
         <dd><strong>c00184_NODE_18..</strong> (original name was: NODE_184_length_7337_cov_7.0595)</dd>
       
         <dd><strong>c00185_NODE_18..</strong> (original name was: NODE_185_length_7226_cov_8.78321)</dd>
       
         <dd><strong>c00186_NODE_18..</strong> (original name was: NODE_186_length_7192_cov_17.3853)</dd>
       
         <dd><strong>c00187_NODE_18..</strong> (original name was: NODE_187_length_7113_cov_15.2037)</dd>
       
         <dd><strong>c00188_NODE_18..</strong> (original name was: NODE_188_length_6983_cov_14.3085)</dd>
       
         <dd><strong>c00189_NODE_18..</strong> (original name was: NODE_189_length_6980_cov_15.351)</dd>
       
         <dd><strong>c00190_NODE_19..</strong> (original name was: NODE_190_length_6924_cov_15.887)</dd>
       
         <dd><strong>c00191_NODE_19..</strong> (original name was: NODE_191_length_6873_cov_13.9976)</dd>
       
         <dd><strong>c00192_NODE_19..</strong> (original name was: NODE_192_length_6806_cov_6.73259)</dd>
       
         <dd><strong>c00193_NODE_19..</strong> (original name was: NODE_193_length_6770_cov_16.0194)</dd>
       
         <dd><strong>c00194_NODE_19..</strong> (original name was: NODE_194_length_6767_cov_17.8598)</dd>
       
         <dd><strong>c00195_NODE_19..</strong> (original name was: NODE_195_length_6660_cov_8.82703)</dd>
       
         <dd><strong>c00196_NODE_19..</strong> (original name was: NODE_196_length_6542_cov_11.1624)</dd>
       
         <dd><strong>c00197_NODE_19..</strong> (original name was: NODE_197_length_6527_cov_16.6253)</dd>
       
         <dd><strong>c00198_NODE_19..</strong> (original name was: NODE_198_length_6373_cov_14.3154)</dd>
       
         <dd><strong>c00200_NODE_20..</strong> (original name was: NODE_200_length_6148_cov_13.7183)</dd>
       
         <dd><strong>c00201_NODE_20..</strong> (original name was: NODE_201_length_6133_cov_8.26057)</dd>
       
         <dd><strong>c00202_NODE_20..</strong> (original name was: NODE_202_length_6098_cov_8.24652)</dd>
       
         <dd><strong>c00203_NODE_20..</strong> (original name was: NODE_203_length_6092_cov_7.51165)</dd>
       
         <dd><strong>c00204_NODE_20..</strong> (original name was: NODE_204_length_6013_cov_12.9001)</dd>
       
         <dd><strong>c00205_NODE_20..</strong> (original name was: NODE_205_length_6005_cov_20.8583)</dd>
       
         <dd><strong>c00206_NODE_20..</strong> (original name was: NODE_206_length_5992_cov_16.8622)</dd>
       
         <dd><strong>c00207_NODE_20..</strong> (original name was: NODE_207_length_5896_cov_15.1491)</dd>
       
         <dd><strong>c00208_NODE_20..</strong> (original name was: NODE_208_length_5887_cov_20.716)</dd>
       
         <dd><strong>c00209_NODE_20..</strong> (original name was: NODE_209_length_5887_cov_27.5755)</dd>
       
         <dd><strong>c00210_NODE_21..</strong> (original name was: NODE_210_length_5848_cov_14.3629)</dd>
       
         <dd><strong>c00211_NODE_21..</strong> (original name was: NODE_211_length_5843_cov_12.397)</dd>
       
         <dd><strong>c00212_NODE_21..</strong> (original name was: NODE_212_length_5804_cov_11.8927)</dd>
       
         <dd><strong>c00213_NODE_21..</strong> (original name was: NODE_213_length_5801_cov_12.8068)</dd>
       
         <dd><strong>c00214_NODE_21..</strong> (original name was: NODE_214_length_5798_cov_14.4442)</dd>
       
         <dd><strong>c00215_NODE_21..</strong> (original name was: NODE_215_length_5787_cov_10.0915)</dd>
       
         <dd><strong>c00216_NODE_21..</strong> (original name was: NODE_216_length_5779_cov_8.17693)</dd>
       
         <dd><strong>c00217_NODE_21..</strong> (original name was: NODE_217_length_5704_cov_19.8573)</dd>
       
         <dd><strong>c00218_NODE_21..</strong> (original name was: NODE_218_length_5615_cov_10.2957)</dd>
       
         <dd><strong>c00219_NODE_21..</strong> (original name was: NODE_219_length_5545_cov_9.80509)</dd>
       
         <dd><strong>c00220_NODE_22..</strong> (original name was: NODE_220_length_5492_cov_13.3087)</dd>
       
         <dd><strong>c00221_NODE_22..</strong> (original name was: NODE_221_length_5480_cov_11.8033)</dd>
       
         <dd><strong>c00222_NODE_22..</strong> (original name was: NODE_222_length_5479_cov_12.8898)</dd>
       
         <dd><strong>c00223_NODE_22..</strong> (original name was: NODE_223_length_5426_cov_19.1863)</dd>
       
         <dd><strong>c00224_NODE_22..</strong> (original name was: NODE_224_length_5355_cov_20.8269)</dd>
       
         <dd><strong>c00225_NODE_22..</strong> (original name was: NODE_225_length_5296_cov_10.3769)</dd>
       
         <dd><strong>c00226_NODE_22..</strong> (original name was: NODE_226_length_5292_cov_16.5915)</dd>
       
         <dd><strong>c00227_NODE_22..</strong> (original name was: NODE_227_length_5260_cov_15.4687)</dd>
       
         <dd><strong>c00228_NODE_22..</strong> (original name was: NODE_228_length_5180_cov_10.9802)</dd>
       
         <dd><strong>c00229_NODE_22..</strong> (original name was: NODE_229_length_5168_cov_11.0865)</dd>
       
         <dd><strong>c00230_NODE_23..</strong> (original name was: NODE_230_length_5077_cov_10.6301)</dd>
       
         <dd><strong>c00231_NODE_23..</strong> (original name was: NODE_231_length_5062_cov_12.9791)</dd>
       
         <dd><strong>c00232_NODE_23..</strong> (original name was: NODE_232_length_5053_cov_16.2722)</dd>
       
         <dd><strong>c00233_NODE_23..</strong> (original name was: NODE_233_length_5035_cov_65.086)</dd>
       
         <dd><strong>c00234_NODE_23..</strong> (original name was: NODE_234_length_5026_cov_20.9933)</dd>
       
         <dd><strong>c00235_NODE_23..</strong> (original name was: NODE_235_length_4991_cov_13.2675)</dd>
       
         <dd><strong>c00236_NODE_23..</strong> (original name was: NODE_236_length_4985_cov_10.41)</dd>
       
         <dd><strong>c00237_NODE_23..</strong> (original name was: NODE_237_length_4956_cov_11.6786)</dd>
       
         <dd><strong>c00238_NODE_23..</strong> (original name was: NODE_238_length_4954_cov_8.62068)</dd>
       
         <dd><strong>c00239_NODE_23..</strong> (original name was: NODE_239_length_4953_cov_11.6024)</dd>
       
         <dd><strong>c00240_NODE_24..</strong> (original name was: NODE_240_length_4856_cov_6.00719)</dd>
       
         <dd><strong>c00241_NODE_24..</strong> (original name was: NODE_241_length_4824_cov_11.0673)</dd>
       
         <dd><strong>c00242_NODE_24..</strong> (original name was: NODE_242_length_4815_cov_9.35751)</dd>
       
         <dd><strong>c00243_NODE_24..</strong> (original name was: NODE_243_length_4802_cov_12.4056)</dd>
       
         <dd><strong>c00244_NODE_24..</strong> (original name was: NODE_244_length_4791_cov_13.2078)</dd>
       
         <dd><strong>c00245_NODE_24..</strong> (original name was: NODE_245_length_4775_cov_8.73602)</dd>
       
         <dd><strong>c00246_NODE_24..</strong> (original name was: NODE_246_length_4769_cov_12.0881)</dd>
       
         <dd><strong>c00247_NODE_24..</strong> (original name was: NODE_247_length_4751_cov_7.66285)</dd>
       
         <dd><strong>c00248_NODE_24..</strong> (original name was: NODE_248_length_4709_cov_7.421)</dd>
       
         <dd><strong>c00249_NODE_24..</strong> (original name was: NODE_249_length_4701_cov_32.761)</dd>
       
         <dd><strong>c00250_NODE_25..</strong> (original name was: NODE_250_length_4605_cov_9.31577)</dd>
       
         <dd><strong>c00251_NODE_25..</strong> (original name was: NODE_251_length_4596_cov_21.7939)</dd>
       
         <dd><strong>c00252_NODE_25..</strong> (original name was: NODE_252_length_4527_cov_22.4539)</dd>
       
         <dd><strong>c00253_NODE_25..</strong> (original name was: NODE_253_length_4526_cov_18.5065)</dd>
       
         <dd><strong>c00254_NODE_25..</strong> (original name was: NODE_254_length_4513_cov_9.0611)</dd>
       
         <dd><strong>c00255_NODE_25..</strong> (original name was: NODE_255_length_4239_cov_15.6576)</dd>
       
         <dd><strong>c00256_NODE_25..</strong> (original name was: NODE_256_length_4433_cov_12.2218)</dd>
       
         <dd><strong>c00257_NODE_25..</strong> (original name was: NODE_257_length_4405_cov_5.10823)</dd>
       
         <dd><strong>c00258_NODE_25..</strong> (original name was: NODE_258_length_4350_cov_20.089)</dd>
       
         <dd><strong>c00259_NODE_25..</strong> (original name was: NODE_259_length_4341_cov_8.6113)</dd>
       
         <dd><strong>c00260_NODE_26..</strong> (original name was: NODE_260_length_4335_cov_18.0411)</dd>
       
         <dd><strong>c00261_NODE_26..</strong> (original name was: NODE_261_length_4319_cov_15.1579)</dd>
       
         <dd><strong>c00262_NODE_26..</strong> (original name was: NODE_262_length_4264_cov_6.49384)</dd>
       
         <dd><strong>c00263_NODE_26..</strong> (original name was: NODE_263_length_4257_cov_13.0262)</dd>
       
         <dd><strong>c00264_NODE_26..</strong> (original name was: NODE_264_length_4234_cov_8.7626)</dd>
       
         <dd><strong>c00265_NODE_26..</strong> (original name was: NODE_265_length_4213_cov_12.3972)</dd>
       
         <dd><strong>c00266_NODE_26..</strong> (original name was: NODE_266_length_4180_cov_6.72415)</dd>
       
         <dd><strong>c00267_NODE_26..</strong> (original name was: NODE_267_length_4166_cov_9.64967)</dd>
       
         <dd><strong>c00268_NODE_26..</strong> (original name was: NODE_268_length_4164_cov_7.61779)</dd>
       
         <dd><strong>c00269_NODE_26..</strong> (original name was: NODE_269_length_4160_cov_8.0305)</dd>
       
         <dd><strong>c00270_NODE_27..</strong> (original name was: NODE_270_length_4143_cov_18.886)</dd>
       
         <dd><strong>c00271_NODE_27..</strong> (original name was: NODE_271_length_4142_cov_4.90785)</dd>
       
         <dd><strong>c00272_NODE_27..</strong> (original name was: NODE_272_length_4135_cov_20.8154)</dd>
       
         <dd><strong>c00273_NODE_27..</strong> (original name was: NODE_273_length_4082_cov_6.71353)</dd>
       
         <dd><strong>c00274_NODE_27..</strong> (original name was: NODE_274_length_4039_cov_24.0514)</dd>
       
         <dd><strong>c00275_NODE_27..</strong> (original name was: NODE_275_length_4018_cov_7.54433)</dd>
       
         <dd><strong>c00276_NODE_27..</strong> (original name was: NODE_276_length_4011_cov_17.1972)</dd>
       
         <dd><strong>c00277_NODE_27..</strong> (original name was: NODE_277_length_3995_cov_11.3976)</dd>
       
         <dd><strong>c00278_NODE_27..</strong> (original name was: NODE_278_length_3970_cov_6.72339)</dd>
       
         <dd><strong>c00279_NODE_27..</strong> (original name was: NODE_279_length_3959_cov_9.95772)</dd>
       
         <dd><strong>c00280_NODE_28..</strong> (original name was: NODE_280_length_3959_cov_52.0955)</dd>
       
         <dd><strong>c00281_NODE_28..</strong> (original name was: NODE_281_length_3957_cov_6.85744)</dd>
       
         <dd><strong>c00282_NODE_28..</strong> (original name was: NODE_282_length_3940_cov_8.58668)</dd>
       
         <dd><strong>c00283_NODE_28..</strong> (original name was: NODE_283_length_3939_cov_12.292)</dd>
       
         <dd><strong>c00284_NODE_28..</strong> (original name was: NODE_284_length_3933_cov_8.26957)</dd>
       
         <dd><strong>c00285_NODE_28..</strong> (original name was: NODE_285_length_3924_cov_7.14564)</dd>
       
         <dd><strong>c00286_NODE_28..</strong> (original name was: NODE_286_length_3864_cov_7.97699)</dd>
       
         <dd><strong>c00287_NODE_28..</strong> (original name was: NODE_287_length_3863_cov_9.76419)</dd>
       
         <dd><strong>c00288_NODE_28..</strong> (original name was: NODE_288_length_3837_cov_13.9606)</dd>
       
         <dd><strong>c00289_NODE_28..</strong> (original name was: NODE_289_length_3834_cov_11.2587)</dd>
       
         <dd><strong>c00290_NODE_29..</strong> (original name was: NODE_290_length_3826_cov_7.8405)</dd>
       
         <dd><strong>c00291_NODE_29..</strong> (original name was: NODE_291_length_3821_cov_19.8671)</dd>
       
         <dd><strong>c00292_NODE_29..</strong> (original name was: NODE_292_length_3811_cov_10.5377)</dd>
       
         <dd><strong>c00293_NODE_29..</strong> (original name was: NODE_293_length_3788_cov_10.6856)</dd>
       
         <dd><strong>c00294_NODE_29..</strong> (original name was: NODE_294_length_3779_cov_14.8773)</dd>
       
         <dd><strong>c00295_NODE_29..</strong> (original name was: NODE_295_length_3770_cov_15.6915)</dd>
       
         <dd><strong>c00296_NODE_29..</strong> (original name was: NODE_296_length_3748_cov_9.67965)</dd>
       
         <dd><strong>c00297_NODE_29..</strong> (original name was: NODE_297_length_3741_cov_9.58439)</dd>
       
         <dd><strong>c00298_NODE_29..</strong> (original name was: NODE_298_length_3741_cov_11.6273)</dd>
       
         <dd><strong>c00299_NODE_29..</strong> (original name was: NODE_299_length_3735_cov_16.3362)</dd>
       
         <dd><strong>c00300_NODE_30..</strong> (original name was: NODE_300_length_3730_cov_13.4097)</dd>
       
         <dd><strong>c00301_NODE_30..</strong> (original name was: NODE_301_length_3707_cov_21.0877)</dd>
       
         <dd><strong>c00302_NODE_30..</strong> (original name was: NODE_302_length_3702_cov_4.6565)</dd>
       
         <dd><strong>c00303_NODE_30..</strong> (original name was: NODE_303_length_3686_cov_11.5445)</dd>
       
         <dd><strong>c00304_NODE_30..</strong> (original name was: NODE_304_length_3672_cov_7.58364)</dd>
       
         <dd><strong>c00305_NODE_30..</strong> (original name was: NODE_305_length_3665_cov_6.66478)</dd>
       
         <dd><strong>c00306_NODE_30..</strong> (original name was: NODE_306_length_3651_cov_5.10045)</dd>
       
         <dd><strong>c00307_NODE_30..</strong> (original name was: NODE_307_length_3612_cov_7.94978)</dd>
       
         <dd><strong>c00308_NODE_30..</strong> (original name was: NODE_308_length_3606_cov_8.27594)</dd>
       
         <dd><strong>c00309_NODE_30..</strong> (original name was: NODE_309_length_3575_cov_12.6879)</dd>
       
         <dd><strong>c00311_NODE_31..</strong> (original name was: NODE_311_length_3514_cov_8.70593)</dd>
       
         <dd><strong>c00312_NODE_31..</strong> (original name was: NODE_312_length_3509_cov_9.30071)</dd>
       
         <dd><strong>c00313_NODE_31..</strong> (original name was: NODE_313_length_3470_cov_8.45648)</dd>
       
         <dd><strong>c00314_NODE_31..</strong> (original name was: NODE_314_length_3417_cov_10.552)</dd>
       
         <dd><strong>c00315_NODE_31..</strong> (original name was: NODE_315_length_3411_cov_4.80207)</dd>
       
         <dd><strong>c00316_NODE_31..</strong> (original name was: NODE_316_length_3403_cov_7.85287)</dd>
       
         <dd><strong>c00317_NODE_31..</strong> (original name was: NODE_317_length_3396_cov_6.62771)</dd>
       
         <dd><strong>c00318_NODE_31..</strong> (original name was: NODE_318_length_3335_cov_26.1983)</dd>
       
         <dd><strong>c00319_NODE_31..</strong> (original name was: NODE_319_length_3317_cov_8.90063)</dd>
       
         <dd><strong>c00320_NODE_32..</strong> (original name was: NODE_320_length_3315_cov_5.44573)</dd>
       
         <dd><strong>c00321_NODE_32..</strong> (original name was: NODE_321_length_3307_cov_19.5186)</dd>
       
         <dd><strong>c00322_NODE_32..</strong> (original name was: NODE_322_length_3297_cov_4.40883)</dd>
       
         <dd><strong>c00323_NODE_32..</strong> (original name was: NODE_323_length_3296_cov_8.23004)</dd>
       
         <dd><strong>c00324_NODE_32..</strong> (original name was: NODE_324_length_3277_cov_8.88857)</dd>
       
         <dd><strong>c00325_NODE_32..</strong> (original name was: NODE_325_length_3227_cov_4.18871)</dd>
       
         <dd><strong>c00326_NODE_32..</strong> (original name was: NODE_326_length_3212_cov_4.92026)</dd>
       
         <dd><strong>c00327_NODE_32..</strong> (original name was: NODE_327_length_3210_cov_8.00551)</dd>
       
         <dd><strong>c00328_NODE_32..</strong> (original name was: NODE_328_length_3207_cov_10.0808)</dd>
       
         <dd><strong>c00329_NODE_32..</strong> (original name was: NODE_329_length_3193_cov_11.699)</dd>
       
         <dd><strong>c00330_NODE_33..</strong> (original name was: NODE_330_length_3187_cov_10.7069)</dd>
       
         <dd><strong>c00331_NODE_33..</strong> (original name was: NODE_331_length_3166_cov_8.97762)</dd>
       
         <dd><strong>c00332_NODE_33..</strong> (original name was: NODE_332_length_3142_cov_11.0395)</dd>
       
         <dd><strong>c00333_NODE_33..</strong> (original name was: NODE_333_length_3135_cov_9.50964)</dd>
       
         <dd><strong>c00334_NODE_33..</strong> (original name was: NODE_334_length_3129_cov_9.93404)</dd>
       
         <dd><strong>c00335_NODE_33..</strong> (original name was: NODE_335_length_3128_cov_13.5182)</dd>
       
         <dd><strong>c00336_NODE_33..</strong> (original name was: NODE_336_length_3109_cov_5.36754)</dd>
       
         <dd><strong>c00337_NODE_33..</strong> (original name was: NODE_337_length_3095_cov_9.3066)</dd>
       
         <dd><strong>c00338_NODE_33..</strong> (original name was: NODE_338_length_3089_cov_5.50945)</dd>
       
         <dd><strong>c00339_NODE_33..</strong> (original name was: NODE_339_length_3088_cov_12.5795)</dd>
       
         <dd><strong>c00340_NODE_34..</strong> (original name was: NODE_340_length_3083_cov_7.00203)</dd>
       
         <dd><strong>c00341_NODE_34..</strong> (original name was: NODE_341_length_3081_cov_69.0085)</dd>
       
         <dd><strong>c00342_NODE_34..</strong> (original name was: NODE_342_length_3080_cov_7.92347)</dd>
       
         <dd><strong>c00343_NODE_34..</strong> (original name was: NODE_343_length_3075_cov_7.42334)</dd>
       
         <dd><strong>c00344_NODE_34..</strong> (original name was: NODE_344_length_3071_cov_11.213)</dd>
       
         <dd><strong>c00345_NODE_34..</strong> (original name was: NODE_345_length_3050_cov_9.67841)</dd>
       
         <dd><strong>c00346_NODE_34..</strong> (original name was: NODE_346_length_3031_cov_6.68113)</dd>
       
         <dd><strong>c00347_NODE_34..</strong> (original name was: NODE_347_length_3018_cov_9.1304)</dd>
       
         <dd><strong>c00348_NODE_34..</strong> (original name was: NODE_348_length_3009_cov_9.38376)</dd>
       
         <dd><strong>c00349_NODE_34..</strong> (original name was: NODE_349_length_2941_cov_7.52878)</dd>
       
         <dd><strong>c00350_NODE_35..</strong> (original name was: NODE_350_length_2941_cov_18.6706)</dd>
       
         <dd><strong>c00351_NODE_35..</strong> (original name was: NODE_351_length_2937_cov_83.4673)</dd>
       
         <dd><strong>c00352_NODE_35..</strong> (original name was: NODE_352_length_2934_cov_4.92483)</dd>
       
         <dd><strong>c00353_NODE_35..</strong> (original name was: NODE_353_length_2910_cov_10.8883)</dd>
       
         <dd><strong>c00354_NODE_35..</strong> (original name was: NODE_354_length_2891_cov_8.61577)</dd>
       
         <dd><strong>c00355_NODE_35..</strong> (original name was: NODE_355_length_2889_cov_91.4718)</dd>
       
         <dd><strong>c00356_NODE_35..</strong> (original name was: NODE_356_length_2885_cov_11.3981)</dd>
       
         <dd><strong>c00357_NODE_35..</strong> (original name was: NODE_357_length_2858_cov_9.77884)</dd>
       
         <dd><strong>c00358_NODE_35..</strong> (original name was: NODE_358_length_2850_cov_7.96181)</dd>
       
         <dd><strong>c00359_NODE_35..</strong> (original name was: NODE_359_length_2846_cov_12.2505)</dd>
       
         <dd><strong>c00360_NODE_36..</strong> (original name was: NODE_360_length_2837_cov_4.03653)</dd>
       
         <dd><strong>c00361_NODE_36..</strong> (original name was: NODE_361_length_2827_cov_7.11926)</dd>
       
         <dd><strong>c00362_NODE_36..</strong> (original name was: NODE_362_length_2801_cov_10.761)</dd>
       
         <dd><strong>c00363_NODE_36..</strong> (original name was: NODE_363_length_2801_cov_10.6657)</dd>
       
         <dd><strong>c00364_NODE_36..</strong> (original name was: NODE_364_length_2800_cov_6.21287)</dd>
       
         <dd><strong>c00365_NODE_36..</strong> (original name was: NODE_365_length_2799_cov_8.0262)</dd>
       
         <dd><strong>c00366_NODE_36..</strong> (original name was: NODE_366_length_2791_cov_11.735)</dd>
       
         <dd><strong>c00367_NODE_36..</strong> (original name was: NODE_367_length_2781_cov_15.4597)</dd>
       
         <dd><strong>c00368_NODE_36..</strong> (original name was: NODE_368_length_2779_cov_10.7187)</dd>
       
         <dd><strong>c00369_NODE_36..</strong> (original name was: NODE_369_length_2765_cov_10.5239)</dd>
       
         <dd><strong>c00370_NODE_37..</strong> (original name was: NODE_370_length_2752_cov_5.85105)</dd>
       
         <dd><strong>c00371_NODE_37..</strong> (original name was: NODE_371_length_2736_cov_7.92794)</dd>
       
         <dd><strong>c00372_NODE_37..</strong> (original name was: NODE_372_length_2734_cov_6.8155)</dd>
       
         <dd><strong>c00373_NODE_37..</strong> (original name was: NODE_373_length_2732_cov_6.49175)</dd>
       
         <dd><strong>c00374_NODE_37..</strong> (original name was: NODE_374_length_2724_cov_7.5645)</dd>
       
         <dd><strong>c00375_NODE_37..</strong> (original name was: NODE_375_length_2719_cov_7.31752)</dd>
       
         <dd><strong>c00376_NODE_37..</strong> (original name was: NODE_376_length_2718_cov_17.4373)</dd>
       
         <dd><strong>c00377_NODE_37..</strong> (original name was: NODE_377_length_2708_cov_12.3045)</dd>
       
         <dd><strong>c00378_NODE_37..</strong> (original name was: NODE_378_length_2693_cov_7.56157)</dd>
       
         <dd><strong>c00379_NODE_37..</strong> (original name was: NODE_379_length_2687_cov_9.57266)</dd>
       
         <dd><strong>c00380_NODE_38..</strong> (original name was: NODE_380_length_2659_cov_6.86453)</dd>
       
         <dd><strong>c00381_NODE_38..</strong> (original name was: NODE_381_length_2652_cov_8.84)</dd>
       
         <dd><strong>c00382_NODE_38..</strong> (original name was: NODE_382_length_2624_cov_7.79335)</dd>
       
         <dd><strong>c00383_NODE_38..</strong> (original name was: NODE_383_length_2611_cov_15.5459)</dd>
       
         <dd><strong>c00384_NODE_38..</strong> (original name was: NODE_384_length_2606_cov_5.35619)</dd>
       
         <dd><strong>c00385_NODE_38..</strong> (original name was: NODE_385_length_2592_cov_5.6288)</dd>
       
         <dd><strong>c00386_NODE_38..</strong> (original name was: NODE_386_length_2582_cov_9.54786)</dd>
       
         <dd><strong>c00387_NODE_38..</strong> (original name was: NODE_387_length_2575_cov_17.058)</dd>
       
         <dd><strong>c00388_NODE_38..</strong> (original name was: NODE_388_length_2548_cov_42.5023)</dd>
       
         <dd><strong>c00389_NODE_38..</strong> (original name was: NODE_389_length_2541_cov_267.176)</dd>
       
         <dd><strong>c00390_NODE_39..</strong> (original name was: NODE_390_length_2535_cov_11.2234)</dd>
       
         <dd><strong>c00391_NODE_39..</strong> (original name was: NODE_391_length_2529_cov_7.78143)</dd>
       
         <dd><strong>c00392_NODE_39..</strong> (original name was: NODE_392_length_2506_cov_6.39176)</dd>
       
         <dd><strong>c00393_NODE_39..</strong> (original name was: NODE_393_length_2495_cov_20.5017)</dd>
       
         <dd><strong>c00394_NODE_39..</strong> (original name was: NODE_394_length_2489_cov_7.66765)</dd>
       
         <dd><strong>c00395_NODE_39..</strong> (original name was: NODE_395_length_2475_cov_14.9847)</dd>
       
         <dd><strong>c00396_NODE_39..</strong> (original name was: NODE_396_length_2463_cov_6.35103)</dd>
       
         <dd><strong>c00397_NODE_39..</strong> (original name was: NODE_397_length_2459_cov_7.0729)</dd>
       
         <dd><strong>c00398_NODE_39..</strong> (original name was: NODE_398_length_2456_cov_11.0511)</dd>
       
         <dd><strong>c00399_NODE_39..</strong> (original name was: NODE_399_length_2447_cov_93.128)</dd>
       
         <dd><strong>c00400_NODE_40..</strong> (original name was: NODE_400_length_2443_cov_5.85147)</dd>
       
         <dd><strong>c00401_NODE_40..</strong> (original name was: NODE_401_length_2419_cov_8.16099)</dd>
       
         <dd><strong>c00402_NODE_40..</strong> (original name was: NODE_402_length_2410_cov_10.8717)</dd>
       
         <dd><strong>c00403_NODE_40..</strong> (original name was: NODE_403_length_2405_cov_13.9039)</dd>
       
         <dd><strong>c00404_NODE_40..</strong> (original name was: NODE_404_length_2402_cov_6.4167)</dd>
       
         <dd><strong>c00405_NODE_40..</strong> (original name was: NODE_405_length_2399_cov_7.17077)</dd>
       
         <dd><strong>c00406_NODE_40..</strong> (original name was: NODE_406_length_2388_cov_8.81911)</dd>
       
         <dd><strong>c00407_NODE_40..</strong> (original name was: NODE_407_length_2383_cov_8.69016)</dd>
       
         <dd><strong>c00408_NODE_40..</strong> (original name was: NODE_408_length_2373_cov_99.7809)</dd>
       
         <dd><strong>c00409_NODE_40..</strong> (original name was: NODE_409_length_2366_cov_10.841)</dd>
       
         <dd><strong>c00410_NODE_41..</strong> (original name was: NODE_410_length_2363_cov_21.0514)</dd>
       
         <dd><strong>c00411_NODE_41..</strong> (original name was: NODE_411_length_2351_cov_10.165)</dd>
       
         <dd><strong>c00412_NODE_41..</strong> (original name was: NODE_412_length_2343_cov_8.338)</dd>
       
         <dd><strong>c00413_NODE_41..</strong> (original name was: NODE_413_length_2335_cov_6.02264)</dd>
       
         <dd><strong>c00414_NODE_41..</strong> (original name was: NODE_414_length_2327_cov_12.3768)</dd>
       
         <dd><strong>c00415_NODE_41..</strong> (original name was: NODE_415_length_2304_cov_11.9237)</dd>
       
         <dd><strong>c00416_NODE_41..</strong> (original name was: NODE_416_length_2292_cov_7.07206)</dd>
       
         <dd><strong>c00417_NODE_41..</strong> (original name was: NODE_417_length_2278_cov_13.1046)</dd>
       
         <dd><strong>c00418_NODE_41..</strong> (original name was: NODE_418_length_2273_cov_10.1636)</dd>
       
         <dd><strong>c00419_NODE_41..</strong> (original name was: NODE_419_length_2266_cov_9.74614)</dd>
       
         <dd><strong>c00420_NODE_42..</strong> (original name was: NODE_420_length_2264_cov_7.02153)</dd>
       
         <dd><strong>c00421_NODE_42..</strong> (original name was: NODE_421_length_2258_cov_12.9305)</dd>
       
         <dd><strong>c00422_NODE_42..</strong> (original name was: NODE_422_length_2257_cov_8.89812)</dd>
       
         <dd><strong>c00423_NODE_42..</strong> (original name was: NODE_423_length_2254_cov_11.7132)</dd>
       
         <dd><strong>c00424_NODE_42..</strong> (original name was: NODE_424_length_2248_cov_8.37529)</dd>
       
         <dd><strong>c00425_NODE_42..</strong> (original name was: NODE_425_length_2241_cov_7.59555)</dd>
       
         <dd><strong>c00426_NODE_42..</strong> (original name was: NODE_426_length_2238_cov_5.80199)</dd>
       
         <dd><strong>c00427_NODE_42..</strong> (original name was: NODE_427_length_2235_cov_6.69497)</dd>
       
         <dd><strong>c00428_NODE_42..</strong> (original name was: NODE_428_length_2233_cov_5.22934)</dd>
       
         <dd><strong>c00429_NODE_42..</strong> (original name was: NODE_429_length_2204_cov_9.08185)</dd>
       
         <dd><strong>c00430_NODE_43..</strong> (original name was: NODE_430_length_2201_cov_19.5072)</dd>
       
         <dd><strong>c00431_NODE_43..</strong> (original name was: NODE_431_length_2185_cov_13.9077)</dd>
       
         <dd><strong>c00432_NODE_43..</strong> (original name was: NODE_432_length_2183_cov_7.47422)</dd>
       
         <dd><strong>c00433_NODE_43..</strong> (original name was: NODE_433_length_2172_cov_9.35844)</dd>
       
         <dd><strong>c00434_NODE_43..</strong> (original name was: NODE_434_length_2143_cov_10.8884)</dd>
       
         <dd><strong>c00435_NODE_43..</strong> (original name was: NODE_435_length_2143_cov_14.7331)</dd>
       
         <dd><strong>c00436_NODE_43..</strong> (original name was: NODE_436_length_2122_cov_11.813)</dd>
       
         <dd><strong>c00437_NODE_43..</strong> (original name was: NODE_437_length_2122_cov_11.197)</dd>
       
         <dd><strong>c00438_NODE_43..</strong> (original name was: NODE_438_length_2111_cov_21.5081)</dd>
       
         <dd><strong>c00439_NODE_43..</strong> (original name was: NODE_439_length_2109_cov_8.54844)</dd>
       
         <dd><strong>c00440_NODE_44..</strong> (original name was: NODE_440_length_2106_cov_8.71602)</dd>
       
         <dd><strong>c00441_NODE_44..</strong> (original name was: NODE_441_length_2098_cov_4.23541)</dd>
       
         <dd><strong>c00442_NODE_44..</strong> (original name was: NODE_442_length_2092_cov_11.3542)</dd>
       
         <dd><strong>c00443_NODE_44..</strong> (original name was: NODE_443_length_2082_cov_12.1683)</dd>
       
         <dd><strong>c00444_NODE_44..</strong> (original name was: NODE_444_length_2074_cov_6.61017)</dd>
       
         <dd><strong>c00445_NODE_44..</strong> (original name was: NODE_445_length_2073_cov_6.67883)</dd>
       
         <dd><strong>c00446_NODE_44..</strong> (original name was: NODE_446_length_2061_cov_8.74664)</dd>
       
         <dd><strong>c00447_NODE_44..</strong> (original name was: NODE_447_length_2059_cov_12.543)</dd>
       
         <dd><strong>c00448_NODE_44..</strong> (original name was: NODE_448_length_2036_cov_14.5946)</dd>
       
         <dd><strong>c00449_NODE_44..</strong> (original name was: NODE_449_length_2009_cov_10.0956)</dd>
       
         <dd><strong>c00450_NODE_45..</strong> (original name was: NODE_450_length_1996_cov_36.7523)</dd>
       
         <dd><strong>c00451_NODE_45..</strong> (original name was: NODE_451_length_1993_cov_14.4496)</dd>
       
         <dd><strong>c00452_NODE_45..</strong> (original name was: NODE_452_length_1970_cov_11.9154)</dd>
       
         <dd><strong>c00453_NODE_45..</strong> (original name was: NODE_453_length_1968_cov_11.465)</dd>
       
         <dd><strong>c00454_NODE_45..</strong> (original name was: NODE_454_length_1962_cov_16.9918)</dd>
       
         <dd><strong>c00455_NODE_45..</strong> (original name was: NODE_455_length_1941_cov_5.02701)</dd>
       
         <dd><strong>c00456_NODE_45..</strong> (original name was: NODE_456_length_1939_cov_6.58885)</dd>
       
         <dd><strong>c00457_NODE_45..</strong> (original name was: NODE_457_length_1938_cov_10.2374)</dd>
       
         <dd><strong>c00458_NODE_45..</strong> (original name was: NODE_458_length_1935_cov_5.14546)</dd>
       
         <dd><strong>c00459_NODE_45..</strong> (original name was: NODE_459_length_1920_cov_4.53653)</dd>
       
         <dd><strong>c00460_NODE_46..</strong> (original name was: NODE_460_length_1905_cov_5.92351)</dd>
       
         <dd><strong>c00461_NODE_46..</strong> (original name was: NODE_461_length_1903_cov_8.81137)</dd>
       
         <dd><strong>c00463_NODE_46..</strong> (original name was: NODE_463_length_1901_cov_11.0496)</dd>
       
         <dd><strong>c00464_NODE_46..</strong> (original name was: NODE_464_length_1899_cov_3.99041)</dd>
       
         <dd><strong>c00465_NODE_46..</strong> (original name was: NODE_465_length_1891_cov_18.3209)</dd>
       
         <dd><strong>c00466_NODE_46..</strong> (original name was: NODE_466_length_1889_cov_8.71283)</dd>
       
         <dd><strong>c00467_NODE_46..</strong> (original name was: NODE_467_length_1888_cov_12.7666)</dd>
       
         <dd><strong>c00468_NODE_46..</strong> (original name was: NODE_468_length_1883_cov_14.594)</dd>
       
         <dd><strong>c00469_NODE_46..</strong> (original name was: NODE_469_length_1880_cov_12.3092)</dd>
       
         <dd><strong>c00470_NODE_47..</strong> (original name was: NODE_470_length_1860_cov_4.82977)</dd>
       
         <dd><strong>c00471_NODE_47..</strong> (original name was: NODE_471_length_1851_cov_4.64385)</dd>
       
         <dd><strong>c00472_NODE_47..</strong> (original name was: NODE_472_length_1840_cov_10.8161)</dd>
       
         <dd><strong>c00473_NODE_47..</strong> (original name was: NODE_473_length_1835_cov_6.50644)</dd>
       
         <dd><strong>c00474_NODE_47..</strong> (original name was: NODE_474_length_1832_cov_8.33138)</dd>
       
         <dd><strong>c00475_NODE_47..</strong> (original name was: NODE_475_length_1825_cov_7.73145)</dd>
       
         <dd><strong>c00476_NODE_47..</strong> (original name was: NODE_476_length_1822_cov_8.50855)</dd>
       
         <dd><strong>c00477_NODE_47..</strong> (original name was: NODE_477_length_1821_cov_7.09091)</dd>
       
         <dd><strong>c00478_NODE_47..</strong> (original name was: NODE_478_length_1814_cov_29.914)</dd>
       
         <dd><strong>c00479_NODE_47..</strong> (original name was: NODE_479_length_1810_cov_14.1557)</dd>
       
         <dd><strong>c00480_NODE_48..</strong> (original name was: NODE_480_length_1808_cov_13.442)</dd>
       
         <dd><strong>c00481_NODE_48..</strong> (original name was: NODE_481_length_1795_cov_3.17026)</dd>
       
         <dd><strong>c00482_NODE_48..</strong> (original name was: NODE_482_length_1791_cov_7.14423)</dd>
       
         <dd><strong>c00483_NODE_48..</strong> (original name was: NODE_483_length_1788_cov_7.31908)</dd>
       
         <dd><strong>c00484_NODE_48..</strong> (original name was: NODE_484_length_1772_cov_4.81337)</dd>
       
         <dd><strong>c00485_NODE_48..</strong> (original name was: NODE_485_length_1765_cov_9.92125)</dd>
       
         <dd><strong>c00486_NODE_48..</strong> (original name was: NODE_486_length_1763_cov_10.7641)</dd>
       
         <dd><strong>c00487_NODE_48..</strong> (original name was: NODE_487_length_1744_cov_6.3692)</dd>
       
         <dd><strong>c00488_NODE_48..</strong> (original name was: NODE_488_length_1726_cov_9.79487)</dd>
       
         <dd><strong>c00489_NODE_48..</strong> (original name was: NODE_489_length_1713_cov_7.65574)</dd>
       
         <dd><strong>c00490_NODE_49..</strong> (original name was: NODE_490_length_1711_cov_6.48232)</dd>
       
         <dd><strong>c00491_NODE_49..</strong> (original name was: NODE_491_length_1709_cov_9.82491)</dd>
       
         <dd><strong>c00492_NODE_49..</strong> (original name was: NODE_492_length_1705_cov_7.01522)</dd>
       
         <dd><strong>c00493_NODE_49..</strong> (original name was: NODE_493_length_1696_cov_10.355)</dd>
       
         <dd><strong>c00494_NODE_49..</strong> (original name was: NODE_494_length_1695_cov_6.27551)</dd>
       
         <dd><strong>c00495_NODE_49..</strong> (original name was: NODE_495_length_1683_cov_7.70951)</dd>
       
         <dd><strong>c00496_NODE_49..</strong> (original name was: NODE_496_length_1680_cov_9.41468)</dd>
       
         <dd><strong>c00497_NODE_49..</strong> (original name was: NODE_497_length_1678_cov_3.804)</dd>
       
         <dd><strong>c00498_NODE_49..</strong> (original name was: NODE_498_length_1666_cov_5.22417)</dd>
       
         <dd><strong>c00499_NODE_49..</strong> (original name was: NODE_499_length_1664_cov_6.1607)</dd>
       
         <dd><strong>c00500_NODE_50..</strong> (original name was: NODE_500_length_1648_cov_7.65089)</dd>
       
         <dd><strong>c00501_NODE_50..</strong> (original name was: NODE_501_length_1623_cov_13.9205)</dd>
       
         <dd><strong>c00502_NODE_50..</strong> (original name was: NODE_502_length_1618_cov_4.35412)</dd>
       
         <dd><strong>c00503_NODE_50..</strong> (original name was: NODE_503_length_1618_cov_9.1053)</dd>
       
         <dd><strong>c00504_NODE_50..</strong> (original name was: NODE_504_length_1602_cov_7.22915)</dd>
       
         <dd><strong>c00505_NODE_50..</strong> (original name was: NODE_505_length_1596_cov_5.98503)</dd>
       
         <dd><strong>c00506_NODE_50..</strong> (original name was: NODE_506_length_1597_cov_6.85034)</dd>
       
         <dd><strong>c00507_NODE_50..</strong> (original name was: NODE_507_length_1593_cov_8.76808)</dd>
       
         <dd><strong>c00508_NODE_50..</strong> (original name was: NODE_508_length_1572_cov_12.8145)</dd>
       
         <dd><strong>c00509_NODE_50..</strong> (original name was: NODE_509_length_1563_cov_4.41226)</dd>
       
         <dd><strong>c00510_NODE_51..</strong> (original name was: NODE_510_length_1562_cov_4.42369)</dd>
       
         <dd><strong>c00511_NODE_51..</strong> (original name was: NODE_511_length_1554_cov_5.98458)</dd>
       
         <dd><strong>c00512_NODE_51..</strong> (original name was: NODE_512_length_1549_cov_7.327)</dd>
       
         <dd><strong>c00513_NODE_51..</strong> (original name was: NODE_513_length_1549_cov_188.819)</dd>
       
         <dd><strong>c00514_NODE_51..</strong> (original name was: NODE_514_length_1548_cov_11.6165)</dd>
       
         <dd><strong>c00515_NODE_51..</strong> (original name was: NODE_515_length_1536_cov_7.86941)</dd>
       
         <dd><strong>c00516_NODE_51..</strong> (original name was: NODE_516_length_1534_cov_10.7079)</dd>
       
         <dd><strong>c00517_NODE_51..</strong> (original name was: NODE_517_length_1527_cov_3.63429)</dd>
       
         <dd><strong>c00518_NODE_51..</strong> (original name was: NODE_518_length_1521_cov_8.94476)</dd>
       
         <dd><strong>c00519_NODE_51..</strong> (original name was: NODE_519_length_1516_cov_5.92153)</dd>
       
         <dd><strong>c00520_NODE_52..</strong> (original name was: NODE_520_length_1515_cov_9.66643)</dd>
       
         <dd><strong>c00521_NODE_52..</strong> (original name was: NODE_521_length_1502_cov_2.33236)</dd>
       
         <dd><strong>c00522_NODE_52..</strong> (original name was: NODE_522_length_1502_cov_16.6175)</dd>
       
         <dd><strong>c00523_NODE_52..</strong> (original name was: NODE_523_length_1496_cov_7.67495)</dd>
       
         <dd><strong>c00524_NODE_52..</strong> (original name was: NODE_524_length_1488_cov_1310.86)</dd>
       
         <dd><strong>c00525_NODE_52..</strong> (original name was: NODE_525_length_1485_cov_4.09647)</dd>
       
         <dd><strong>c00526_NODE_52..</strong> (original name was: NODE_526_length_1482_cov_5.85756)</dd>
       
         <dd><strong>c00527_NODE_52..</strong> (original name was: NODE_527_length_1478_cov_9.60252)</dd>
       
         <dd><strong>c00528_NODE_52..</strong> (original name was: NODE_528_length_1476_cov_41.212)</dd>
       
         <dd><strong>c00529_NODE_52..</strong> (original name was: NODE_529_length_1468_cov_5.03952)</dd>
       
         <dd><strong>c00530_NODE_53..</strong> (original name was: NODE_530_length_1461_cov_6.08171)</dd>
       
         <dd><strong>c00531_NODE_53..</strong> (original name was: NODE_531_length_1445_cov_4.7739)</dd>
       
         <dd><strong>c00533_NODE_53..</strong> (original name was: NODE_533_length_1403_cov_15.5721)</dd>
       
         <dd><strong>c00534_NODE_53..</strong> (original name was: NODE_534_length_1401_cov_6.02512)</dd>
       
         <dd><strong>c00535_NODE_53..</strong> (original name was: NODE_535_length_1396_cov_12.0378)</dd>
       
         <dd><strong>c00536_NODE_53..</strong> (original name was: NODE_536_length_1378_cov_4.56435)</dd>
       
         <dd><strong>c00537_NODE_53..</strong> (original name was: NODE_537_length_1373_cov_12.1413)</dd>
       
         <dd><strong>c00538_NODE_53..</strong> (original name was: NODE_538_length_1372_cov_4.00723)</dd>
       
         <dd><strong>c00539_NODE_53..</strong> (original name was: NODE_539_length_1371_cov_7.45981)</dd>
       
         <dd><strong>c00540_NODE_54..</strong> (original name was: NODE_540_length_1365_cov_9.59532)</dd>
       
         <dd><strong>c00541_NODE_54..</strong> (original name was: NODE_541_length_1355_cov_14.6604)</dd>
       
         <dd><strong>c00542_NODE_54..</strong> (original name was: NODE_542_length_1353_cov_9.15253)</dd>
       
         <dd><strong>c00543_NODE_54..</strong> (original name was: NODE_543_length_1348_cov_8.69533)</dd>
       
         <dd><strong>c00544_NODE_54..</strong> (original name was: NODE_544_length_1341_cov_5.92339)</dd>
       
         <dd><strong>c00545_NODE_54..</strong> (original name was: NODE_545_length_1340_cov_9.47733)</dd>
       
         <dd><strong>c00546_NODE_54..</strong> (original name was: NODE_546_length_1331_cov_4.99336)</dd>
       
         <dd><strong>c00547_NODE_54..</strong> (original name was: NODE_547_length_1329_cov_8.7812)</dd>
       
         <dd><strong>c00548_NODE_54..</strong> (original name was: NODE_548_length_1324_cov_9.34169)</dd>
       
         <dd><strong>c00549_NODE_54..</strong> (original name was: NODE_549_length_1316_cov_5.14802)</dd>
       
         <dd><strong>c00550_NODE_55..</strong> (original name was: NODE_550_length_1300_cov_5.91304)</dd>
       
         <dd><strong>c00551_NODE_55..</strong> (original name was: NODE_551_length_1293_cov_1098.33)</dd>
       
         <dd><strong>c00552_NODE_55..</strong> (original name was: NODE_552_length_1292_cov_4.9382)</dd>
       
         <dd><strong>c00553_NODE_55..</strong> (original name was: NODE_553_length_1292_cov_6.86438)</dd>
       
         <dd><strong>c00554_NODE_55..</strong> (original name was: NODE_554_length_1289_cov_5.45697)</dd>
       
         <dd><strong>c00555_NODE_55..</strong> (original name was: NODE_555_length_1284_cov_4.71824)</dd>
       
         <dd><strong>c00556_NODE_55..</strong> (original name was: NODE_556_length_1274_cov_6.8082)</dd>
       
         <dd><strong>c00557_NODE_55..</strong> (original name was: NODE_557_length_1267_cov_3.9)</dd>
       
         <dd><strong>c00558_NODE_55..</strong> (original name was: NODE_558_length_1258_cov_5.04244)</dd>
       
         <dd><strong>c00559_NODE_55..</strong> (original name was: NODE_559_length_1244_cov_3.72784)</dd>
       
         <dd><strong>c00560_NODE_56..</strong> (original name was: NODE_560_length_1238_cov_2.89019)</dd>
       
         <dd><strong>c00561_NODE_56..</strong> (original name was: NODE_561_length_1233_cov_10.7839)</dd>
       
         <dd><strong>c00562_NODE_56..</strong> (original name was: NODE_562_length_1228_cov_12.6494)</dd>
       
         <dd><strong>c00563_NODE_56..</strong> (original name was: NODE_563_length_1225_cov_13.5856)</dd>
       
         <dd><strong>c00564_NODE_56..</strong> (original name was: NODE_564_length_1223_cov_4.79745)</dd>
       
         <dd><strong>c00565_NODE_56..</strong> (original name was: NODE_565_length_1219_cov_5.96429)</dd>
       
         <dd><strong>c00566_NODE_56..</strong> (original name was: NODE_566_length_1215_cov_4.83456)</dd>
       
         <dd><strong>c00567_NODE_56..</strong> (original name was: NODE_567_length_1213_cov_6.2919)</dd>
       
         <dd><strong>c00568_NODE_56..</strong> (original name was: NODE_568_length_1213_cov_5.8361)</dd>
       
         <dd><strong>c00569_NODE_56..</strong> (original name was: NODE_569_length_1210_cov_5.86242)</dd>
       
         <dd><strong>c00570_NODE_57..</strong> (original name was: NODE_570_length_1200_cov_2.81733)</dd>
       
         <dd><strong>c00571_NODE_57..</strong> (original name was: NODE_571_length_1187_cov_3.36509)</dd>
       
         <dd><strong>c00572_NODE_57..</strong> (original name was: NODE_572_length_1168_cov_6.41499)</dd>
       
         <dd><strong>c00573_NODE_57..</strong> (original name was: NODE_573_length_1157_cov_13.0272)</dd>
       
         <dd><strong>c00574_NODE_57..</strong> (original name was: NODE_574_length_1149_cov_7.42857)</dd>
       
         <dd><strong>c00575_NODE_57..</strong> (original name was: NODE_575_length_1147_cov_5.52941)</dd>
       
         <dd><strong>c00576_NODE_57..</strong> (original name was: NODE_576_length_1142_cov_4.83153)</dd>
       
         <dd><strong>c00577_NODE_57..</strong> (original name was: NODE_577_length_1141_cov_5.76627)</dd>
       
         <dd><strong>c00578_NODE_57..</strong> (original name was: NODE_578_length_1139_cov_6.70158)</dd>
       
         <dd><strong>c00579_NODE_57..</strong> (original name was: NODE_579_length_1139_cov_6.64625)</dd>
       
         <dd><strong>c00580_NODE_58..</strong> (original name was: NODE_580_length_1136_cov_3.22101)</dd>
       
         <dd><strong>c00581_NODE_58..</strong> (original name was: NODE_581_length_1131_cov_4.58367)</dd>
       
         <dd><strong>c00582_NODE_58..</strong> (original name was: NODE_582_length_1130_cov_5.38784)</dd>
       
         <dd><strong>c00583_NODE_58..</strong> (original name was: NODE_583_length_1119_cov_10.1704)</dd>
       
         <dd><strong>c00584_NODE_58..</strong> (original name was: NODE_584_length_1115_cov_4.19332)</dd>
       
         <dd><strong>c00585_NODE_58..</strong> (original name was: NODE_585_length_1103_cov_6.17111)</dd>
       
         <dd><strong>c00586_NODE_58..</strong> (original name was: NODE_586_length_1095_cov_4.35227)</dd>
       
         <dd><strong>c00587_NODE_58..</strong> (original name was: NODE_587_length_1085_cov_4.63152)</dd>
       
         <dd><strong>c00588_NODE_58..</strong> (original name was: NODE_588_length_1079_cov_4.26366)</dd>
       
         <dd><strong>c00589_NODE_58..</strong> (original name was: NODE_589_length_1078_cov_7.08202)</dd>
       
         <dd><strong>c00590_NODE_59..</strong> (original name was: NODE_590_length_1074_cov_8.14889)</dd>
       
         <dd><strong>c00591_NODE_59..</strong> (original name was: NODE_591_length_1065_cov_3.87953)</dd>
       
         <dd><strong>c00592_NODE_59..</strong> (original name was: NODE_592_length_1052_cov_324.385)</dd>
       
         <dd><strong>c00593_NODE_59..</strong> (original name was: NODE_593_length_1052_cov_7.85081)</dd>
       
         <dd><strong>c00594_NODE_59..</strong> (original name was: NODE_594_length_1031_cov_4.01881)</dd>
       
         <dd><strong>c00595_NODE_59..</strong> (original name was: NODE_595_length_1029_cov_4.78936)</dd>
       
         <dd><strong>c00596_NODE_59..</strong> (original name was: NODE_596_length_1027_cov_3.21)</dd>
       
         <dd><strong>c00597_NODE_59..</strong> (original name was: NODE_597_length_1017_cov_4.78876)</dd>
       
         <dd><strong>c00598_NODE_59..</strong> (original name was: NODE_598_length_1016_cov_4.42295)</dd>
       
         <dd><strong>c00599_NODE_59..</strong> (original name was: NODE_599_length_1011_cov_3.67986)</dd>
       
         <dd><strong>c00600_NODE_60..</strong> (original name was: NODE_600_length_1009_cov_10.4286)</dd>
       
     </dl>
     
    </div>
    
    <div class="overview-switches">
     <div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    
   </div>
  </div>
  






<div class="page" id='r6c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00006_NODE_6_.. - Region 1 - NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-44"></div> <span class="help-tooltip" id="region_svg-help-44">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00006_NODE_6_...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r6c1-svg" data-filename="c00006_NODE_6_.._r6c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 3,118 - 46,699 nt. (total: 43,582 nt)
        
        <a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r6c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r6c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-45"></div> <span class="help-tooltip" id="focus-panel-help-45">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r6c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r6c1-nrps-pks-bubbles" data-name="r6c1-nrps-pks-bubbles">
        NRPS/PKS modules
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r6c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-1"></div> <span class="help-tooltip" id="nrps-domain-help-1">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-details-svg" data-filename="c00006_NODE_6_.._r6c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r6c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r6c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r6c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r6c1-nrps-pks-bubbles">
      <div class="details">
    <div class="heading">
      <span>Module view</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-6"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-6">Shows module structures for each candidate in <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>
                     regions. Genes are shown in predicted order, and are only present when containing at least one complete module.
                     A domain glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r6c1-domain-bubble-svg" data-filename="c00006_NODE_6_.._r6c1_domain_bubbles.svg">
        </div>
      </div>
    </div>
    <select id="r6c1-domain-bubble-select" class="nrps-bubble-selector">
    
      
        <option value="CC1">
          Candidate 1 (3117 - 46699): single NRPS-like
        </option>
      
    
    </select>
    
    <div class="details-svg" id="r6c1-domain-bubble-svg-container" style="overflow:auto;">
    </div>
    <div class="bubble-legend-title">Legend</div>
    
    <div class="bubble-legend-container" id="r6c1-bubble-legend"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
      <div class="sidepanel-details-header r6c1-nrps_pks_products" data-name="r6c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r6c1-nrps_pks" data-name="r6c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
    </div>
    
     
    
     
    
     
    <div class="sidepanel-details-section r6c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-4"></div> <span class="help-tooltip" id="nrps-structures-help-4">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 3117 - 46699:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r6c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r6c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r6c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-5"></div> <span class="help-tooltip" id="nrps-monomers-help-5">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">ctg6_18</span></strong>: X
      <div class="collapser collapser-target-ctg6_18 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (225..621)</strong>: X
          <div class="collapser collapser-target-ctg6_18 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-ctg6_18-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAWaaGvVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>leu</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r25c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00025_NODE_25.. - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-46"></div> <span class="help-tooltip" id="region_svg-help-46">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00025_NODE_25...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r25c1-svg" data-filename="c00025_NODE_25.._r25c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 9,528 - 31,381 nt. (total: 21,854 nt)
        
        <a class="cluster-rules-header" id="r25c1-rules-header" href="#r25c1">Show pHMM detection rules used</a>
        
        
      </div>
      
      <div class="cluster-rules" id="r25c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>
      
      <div class="region-svg-container">
       <div id='r25c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-47"></div> <span class="help-tooltip" id="focus-panel-help-47">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r25c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r25c1-knownclusterblast" data-name="r25c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r25c1-subclusterblast" data-name="r25c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r25c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-7"></div> <span class="help-tooltip" id="cb-known-help-7">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r25c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r25c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r25c1_1.svg">
        molybdenum cofactor
      </option>
      
    </select>
    <button id="knownclusterblast-r25c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r25c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r25c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-8"></div> <span class="help-tooltip" id="cb-sub-help-8">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
    </div>
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r44c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00044_NODE_44.. - Region 1 - LAP,RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-48"></div> <span class="help-tooltip" id="region_svg-help-48">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00044_NODE_44...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r44c1-svg" data-filename="c00044_NODE_44.._r44c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 20,062 nt. (total: 20,062 nt)
        
        <a class="cluster-rules-header" id="r44c1-rules-header" href="#r44c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r44c1-rules">LAP: ((((goadsporin_like or PF00881 or TIGR03605) and (YcaO or TIGR03882 or TIGR03604)) or TIGR03603 or TIGR03602) and not cyanobactin_synth)<br>RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r44c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-49"></div> <span class="help-tooltip" id="focus-panel-help-49">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r44c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r44c1-knownclusterblast" data-name="r44c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r44c1-subclusterblast" data-name="r44c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
    </div>
    
     
    <div class="body-details-section r44c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-9"></div> <span class="help-tooltip" id="cb-known-help-9">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r44c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-10"></div> <span class="help-tooltip" id="cb-sub-help-10">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
      <div class="sidepanel-details-header r44c1-RREfinder" data-name="r44c1-RREfinder">
        RREFinder
      </div>
     
    
    </div>
    
     
    
     
    <div class="sidepanel-details-section r44c1-RREfinder">
      <div class="more-details">
  <div class="heading">
    <span>RRE predictions</span>
    <div class="help-container"> <div class="help-icon" data-id="rre-side-help-11"></div> <span class="help-tooltip" id="rre-side-help-11">RREfinder results sidepanel.</span></div>
  </div>
  <dl class = 'prediction-text'>
    
      <span><strong>LAP protocluster</strong> (0...20062)</span>
      <dt></dt>
      
      
      
        
        <strong>ctg44_3</strong> <div class="collapser collapser-target-ctg44_3 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Heterocycloanthracin_C_RRE</dd>
          <dd>Location: 0 - 100</dd>
          <dd>Score: 184.0</dd>
        
          <dd>RRE type: Thiopeptide_F_RRE</dd>
          <dd>Location: 10 - 89</dd>
          <dd>Score: 40.5</dd>
        
        </div><br>
      <br>
    
      <span><strong>RiPP-like protocluster</strong> (0...11559)</span>
      <dt></dt>
      
      
      
        
        <strong>ctg44_3</strong> <div class="collapser collapser-target-ctg44_3 collapser-level-cds"> </div><div class="collapser-content">
        
          <dd>RRE type: Heterocycloanthracin_C_RRE</dd>
          <dd>Location: 0 - 100</dd>
          <dd>Score: 184.0</dd>
        
          <dd>RRE type: Thiopeptide_F_RRE</dd>
          <dd>Location: 10 - 89</dd>
          <dd>Score: 40.5</dd>
        
        </div><br>
      <br>
    
    </dl>
</div>
    </div>
     
    
  </div>

 </div>

</div>




<div class="page" id='r49c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00049_NODE_49.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-50"></div> <span class="help-tooltip" id="region_svg-help-50">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00049_NODE_49...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r49c1-svg" data-filename="c00049_NODE_49.._r49c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 25,382 nt. (total: 25,382 nt)
        
        <a class="cluster-rules-header" id="r49c1-rules-header" href="#r49c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r49c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r49c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-51"></div> <span class="help-tooltip" id="focus-panel-help-51">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r49c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r49c1-nrps_pks" data-name="r49c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r49c1-knownclusterblast" data-name="r49c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r49c1-subclusterblast" data-name="r49c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r49c1-nrps-pks-bubbles" data-name="r49c1-nrps-pks-bubbles">
        NRPS/PKS modules
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r49c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-12"></div> <span class="help-tooltip" id="nrps-domain-help-12">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r49c1-details-svg" data-filename="c00049_NODE_49.._r49c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r49c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r49c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-13"></div> <span class="help-tooltip" id="cb-known-help-13">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r49c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r49c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_1.svg">
        nostocyclopeptide A2
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_2.svg">
        gramicidin
      </option>
      
      <option value = "svg/knownclusterblast_r49c1_3.svg">
        anabaenopeptin NZ857 / nostamide A
      </option>
      
    </select>
    <button id="knownclusterblast-r49c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r49c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r49c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-14"></div> <span class="help-tooltip" id="cb-sub-help-14">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r49c1-nrps-pks-bubbles">
      <div class="details">
    <div class="heading">
      <span>Module view</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-17"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-17">Shows module structures for each candidate in <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>
                     regions. Genes are shown in predicted order, and are only present when containing at least one complete module.
                     A domain glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r49c1-domain-bubble-svg" data-filename="c00049_NODE_49.._r49c1_domain_bubbles.svg">
        </div>
      </div>
    </div>
    <select id="r49c1-domain-bubble-select" class="nrps-bubble-selector">
    
      
        <option value="CC1">
          Candidate 1 (0 - 25382): single NRPS
        </option>
      
    
    </select>
    
    <div class="details-svg" id="r49c1-domain-bubble-svg-container" style="overflow:auto;">
    </div>
    <div class="bubble-legend-title">Legend</div>
    
    <div class="bubble-legend-container" id="r49c1-bubble-legend"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
      <div class="sidepanel-details-header r49c1-nrps_pks_products" data-name="r49c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r49c1-nrps_pks" data-name="r49c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
    </div>
    
     
    
     
    
     
    <div class="sidepanel-details-section r49c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-15"></div> <span class="help-tooltip" id="nrps-structures-help-15">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 25382:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r49c1-1" class="smiles-canvas" data-smiles="NC(C(O)C)C(=O)NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC(CCC(=O)N)C(=O)NC(CCC(=O)N)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(D-thr) + (X - ala) + (ser - leu - D-gln - gln)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-thr&amp;nrps2=x,ala&amp;nrps3=ser,leu,d-gln,gln" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-thr*&amp;nrps2=x,ala*&amp;nrps3=ser*,leu*,d-gln*,gln*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r49c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(C(O)C)C(=O)NC([*])C(=O)NC(C)C(=O)NC(CO)C(=O)NC(CC(C)C)C(=O)NC(CCC(=O)N)C(=O)NC(CCC(=O)N)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r49c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-16"></div> <span class="help-tooltip" id="nrps-monomers-help-16">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">ctg49_1</span></strong>: ser - leu - gln - gln
      <div class="collapser collapser-target-ctg49_1 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser,leu,gln,gln" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*,leu*,gln*,gln*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (496..901)</strong>: ser
          <div class="collapser collapser-target-ctg49_1 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ser
              <div class="collapser collapser-target-ctg49_1-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>ser</dd>
   <dt>Single AA prediction:</dt>
   <dd>ser</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DVWHFSLVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ser</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1558..1945)</strong>: leu
          <div class="collapser collapser-target-ctg49_1 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  leu
              <div class="collapser collapser-target-ctg49_1-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>val, leu, ile, abu, iva</dd>
   <dt>Single AA prediction:</dt>
   <dd>leu</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DsQFIgqVVK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>pro</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (2600..3000)</strong>: gln
          <div class="collapser collapser-target-ctg49_1 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gln
              <div class="collapser collapser-target-ctg49_1-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DAiDLGVVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>90% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (4115..4515)</strong>: gln
          <div class="collapser collapser-target-ctg49_1 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gln
              <div class="collapser collapser-target-ctg49_1-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>N/A</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DviDLGVVDK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>80% (moderate)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">ctg49_2</span></strong>: X - ala
      <div class="collapser collapser-target-ctg49_2 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X,ala" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*,ala*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (466..870)</strong>: X
          <div class="collapser collapser-target-ctg49_2 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-ctg49_2-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DvWQFGfIyK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gln</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>70% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1527..1919)</strong>: ala
          <div class="collapser collapser-target-ctg49_2 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  ala
              <div class="collapser collapser-target-ctg49_2-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>ala</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLFNNALTYK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>ala</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">ctg49_4</span></strong>: thr
      <div class="collapser collapser-target-ctg49_4 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (148..557)</strong>: thr
          <div class="collapser collapser-target-ctg49_4 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-ctg49_4-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNIGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r74c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00074_NODE_74.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-52"></div> <span class="help-tooltip" id="region_svg-help-52">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00074_NODE_74...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r74c1-svg" data-filename="c00074_NODE_74.._r74c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 18,380 nt. (total: 18,380 nt)
        
        <a class="cluster-rules-header" id="r74c1-rules-header" href="#r74c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r74c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r74c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-53"></div> <span class="help-tooltip" id="focus-panel-help-53">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r74c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r74c1-nrps_pks" data-name="r74c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r74c1-knownclusterblast" data-name="r74c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r74c1-subclusterblast" data-name="r74c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r74c1-nrps-pks-bubbles" data-name="r74c1-nrps-pks-bubbles">
        NRPS/PKS modules
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r74c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-18"></div> <span class="help-tooltip" id="nrps-domain-help-18">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r74c1-details-svg" data-filename="c00074_NODE_74.._r74c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r74c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r74c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-19"></div> <span class="help-tooltip" id="cb-known-help-19">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r74c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r74c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_1.svg">
        bacillibactin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_2.svg">
        bacillibactin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_3.svg">
        paenibactin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_4.svg">
        streptobactin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_5.svg">
        myxochelin A / myxochelin B
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_6.svg">
        fuscachelin A / fuscachelin B / fuscachelin C
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_7.svg">
        atratumycin
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_8.svg">
        acyldepsipeptide 1
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_9.svg">
        salinamide A / salinamide B / salinamide C / salinamide D / salinamide E / salinamide F / desmethylsalinamide C / desmethylsalinamide E
      </option>
      
      <option value = "svg/knownclusterblast_r74c1_10.svg">
        salinamide A
      </option>
      
    </select>
    <button id="knownclusterblast-r74c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r74c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r74c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-20"></div> <span class="help-tooltip" id="cb-sub-help-20">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r74c1-nrps-pks-bubbles">
      <div class="details">
    <div class="heading">
      <span>Module view</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-23"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-23">Shows module structures for each candidate in <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>
                     regions. Genes are shown in predicted order, and are only present when containing at least one complete module.
                     A domain glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r74c1-domain-bubble-svg" data-filename="c00074_NODE_74.._r74c1_domain_bubbles.svg">
        </div>
      </div>
    </div>
    <select id="r74c1-domain-bubble-select" class="nrps-bubble-selector">
    
      
        <option value="CC1">
          Candidate 1 (0 - 18380): single NRPS
        </option>
      
    
    </select>
    
    <div class="details-svg" id="r74c1-domain-bubble-svg-container" style="overflow:auto;">
    </div>
    <div class="bubble-legend-title">Legend</div>
    
    <div class="bubble-legend-container" id="r74c1-bubble-legend"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
      <div class="sidepanel-details-header r74c1-nrps_pks_products" data-name="r74c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r74c1-nrps_pks" data-name="r74c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
    </div>
    
     
    
     
    
     
    <div class="sidepanel-details-section r74c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-21"></div> <span class="help-tooltip" id="nrps-structures-help-21">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 18380:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r74c1-1" class="smiles-canvas" data-smiles="NCC(=O)NC(C(O)C)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(gly - thr)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,thr" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,thr*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r74c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(C(O)C)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r74c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-22"></div> <span class="help-tooltip" id="nrps-monomers-help-22">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">ctg74_9</span></strong>: gly - thr
      <div class="collapser collapser-target-ctg74_9 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly,thr" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,thr*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
          <dd><strong>AMP-binding (464..864)</strong>: gly
          <div class="collapser collapser-target-ctg74_9 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  gly
              <div class="collapser collapser-target-ctg74_9-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>gly, ala, val, leu, ile, abu, iva</dd>
   <dt>Small clusters prediction:</dt>
   <dd>gly, ala</dd>
   <dt>Single AA prediction:</dt>
   <dd>gly</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILQLGLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
        
          <dd><strong>AMP-binding (1524..1937)</strong>: thr
          <div class="collapser collapser-target-ctg74_9 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  thr
              <div class="collapser collapser-target-ctg74_9-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>ser, thr, dhpg, hpg</dd>
   <dt>Small clusters prediction:</dt>
   <dd>thr</dd>
   <dt>Single AA prediction:</dt>
   <dd>thr</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DFWNVGMVHK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>thr</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
      <strong><span class="serif">ctg74_11</span></strong>: dhb
      <div class="collapser collapser-target-ctg74_11 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=diOH-Bz*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (31..435)</strong>: dhb
          <div class="collapser collapser-target-ctg74_11 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  dhb
              <div class="collapser collapser-target-ctg74_11-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aromatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Small clusters prediction:</dt>
   <dd>dhb, sal</dd>
   <dt>Single AA prediction:</dt>
   <dd>dhb</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">PMPAQGVVNK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>dhb</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r93c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00093_NODE_93.. - Region 1 - siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-54"></div> <span class="help-tooltip" id="region_svg-help-54">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00093_NODE_93...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r93c1-svg" data-filename="c00093_NODE_93.._r93c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 12,527 nt. (total: 12,527 nt)
        
        <a class="cluster-rules-header" id="r93c1-rules-header" href="#r93c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r93c1-rules">siderophore: IucA_IucC</div>
      
      <div class="region-svg-container">
       <div id='r93c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-55"></div> <span class="help-tooltip" id="focus-panel-help-55">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r93c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r93c1-knownclusterblast" data-name="r93c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r93c1-subclusterblast" data-name="r93c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r93c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-24"></div> <span class="help-tooltip" id="cb-known-help-24">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r93c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r93c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r93c1_1.svg">
        petrobactin
      </option>
      
    </select>
    <button id="knownclusterblast-r93c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r93c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r93c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-25"></div> <span class="help-tooltip" id="cb-sub-help-25">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
    </div>
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r138c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00138_NODE_13.. - Region 1 - betalactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-56"></div> <span class="help-tooltip" id="region_svg-help-56">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00138_NODE_13...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r138c1-svg" data-filename="c00138_NODE_13.._r138c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 9,692 nt. (total: 9,692 nt)
        
        <a class="cluster-rules-header" id="r138c1-rules-header" href="#r138c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r138c1-rules">betalactone: (HMGL-like and AMP-binding)</div>
      
      <div class="region-svg-container">
       <div id='r138c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-57"></div> <span class="help-tooltip" id="focus-panel-help-57">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r138c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r138c1-knownclusterblast" data-name="r138c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r138c1-subclusterblast" data-name="r138c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r138c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-26"></div> <span class="help-tooltip" id="cb-known-help-26">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <select id="knownclusterblast-r138c1-select" class="clusterblast-selector">
      <option value="svg/knownclusterblast_r138c1_all.svg">
        All hits
      </option>
      
      <option value = "svg/knownclusterblast_r138c1_1.svg">
        fengycin
      </option>
      
      <option value = "svg/knownclusterblast_r138c1_2.svg">
        mycosubtilin
      </option>
      
      <option value = "svg/knownclusterblast_r138c1_3.svg">
        plipastatin
      </option>
      
    </select>
    <button id="knownclusterblast-r138c1-download">Download graphic</button>
  </div>
  <div id="knownclusterblast-r138c1-svg"></div>
  
</div>
    </div>
     
    <div class="body-details-section r138c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-27"></div> <span class="help-tooltip" id="cb-sub-help-27">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
    </div>
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r199c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00199_NODE_19.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-58"></div> <span class="help-tooltip" id="region_svg-help-58">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00199_NODE_19...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r199c1-svg" data-filename="c00199_NODE_19.._r199c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 6,251 nt. (total: 6,251 nt)
        
        <a class="cluster-rules-header" id="r199c1-rules-header" href="#r199c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r199c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r199c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-59"></div> <span class="help-tooltip" id="focus-panel-help-59">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r199c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r199c1-nrps_pks" data-name="r199c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r199c1-knownclusterblast" data-name="r199c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r199c1-subclusterblast" data-name="r199c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r199c1-nrps-pks-bubbles" data-name="r199c1-nrps-pks-bubbles">
        NRPS/PKS modules
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r199c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-28"></div> <span class="help-tooltip" id="nrps-domain-help-28">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r199c1-details-svg" data-filename="c00199_NODE_19.._r199c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r199c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r199c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-29"></div> <span class="help-tooltip" id="cb-known-help-29">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r199c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-30"></div> <span class="help-tooltip" id="cb-sub-help-30">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r199c1-nrps-pks-bubbles">
      <div class="details">
    <div class="heading">
      <span>Module view</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-33"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-33">Shows module structures for each candidate in <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>
                     regions. Genes are shown in predicted order, and are only present when containing at least one complete module.
                     A domain glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r199c1-domain-bubble-svg" data-filename="c00199_NODE_19.._r199c1_domain_bubbles.svg">
        </div>
      </div>
    </div>
    <select id="r199c1-domain-bubble-select" class="nrps-bubble-selector">
    
      
        <option value="CC1">
          Candidate 1 (0 - 6251): single NRPS
        </option>
      
    
    </select>
    
    <div class="details-svg" id="r199c1-domain-bubble-svg-container" style="overflow:auto;">
    </div>
    <div class="bubble-legend-title">Legend</div>
    
    <div class="bubble-legend-container" id="r199c1-bubble-legend"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
      <div class="sidepanel-details-header r199c1-nrps_pks_products" data-name="r199c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r199c1-nrps_pks" data-name="r199c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
    </div>
    
     
    
     
    
     
    <div class="sidepanel-details-section r199c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-31"></div> <span class="help-tooltip" id="nrps-structures-help-31">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 6251:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r199c1-1" class="smiles-canvas" data-smiles="NC(CS)C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(cys)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r199c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r199c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-32"></div> <span class="help-tooltip" id="nrps-monomers-help-32">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">ctg199_2</span></strong>: cys
      <div class="collapser collapser-target-ctg199_2 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
      
        
      
        
          <dd><strong>AMP-binding (539..919)</strong>: cys
          <div class="collapser collapser-target-ctg199_2 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  cys
              <div class="collapser collapser-target-ctg199_2-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Small clusters prediction:</dt>
   <dd>cys</dd>
   <dt>Single AA prediction:</dt>
   <dd>cys</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DLYNLSLIWK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>cys</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>100% (strong)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r310c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00310_NODE_31.. - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-60"></div> <span class="help-tooltip" id="region_svg-help-60">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00310_NODE_31...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r310c1-svg" data-filename="c00310_NODE_31.._r310c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 3,541 nt. (total: 3,541 nt)
        
        <a class="cluster-rules-header" id="r310c1-rules-header" href="#r310c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r310c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>
      
      <div class="region-svg-container">
       <div id='r310c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-61"></div> <span class="help-tooltip" id="focus-panel-help-61">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r310c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r310c1-nrps_pks" data-name="r310c1-nrps_pks">
        NRPS/PKS domains
      </div>
     
    
     
      <div class="body-details-header r310c1-knownclusterblast" data-name="r310c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r310c1-subclusterblast" data-name="r310c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
     
      <div class="body-details-header r310c1-nrps-pks-bubbles" data-name="r310c1-nrps-pks-bubbles">
        NRPS/PKS modules
      </div>
     
    
    </div>
    
     
    <div class="body-details-section r310c1-nrps_pks">
      <div class="details">
    <div class="heading">
      <span>Detailed domain annotation</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-34"></div> <span class="help-tooltip" id="nrps-domain-help-34">Shows <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r310c1-details-svg" data-filename="c00310_NODE_31.._r310c1_nrps_pks.svg">
        </div>
      </div>
    </div>
    <div class="nrps-pks-domain-buttons">
      <div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
      <div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
    </div>
    <div class="details-svg" id="r310c1-details-svg"></div>
</div>
    </div>
     
    
     
    <div class="body-details-section r310c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-35"></div> <span class="help-tooltip" id="cb-known-help-35">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r310c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-36"></div> <span class="help-tooltip" id="cb-sub-help-36">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
     
    <div class="body-details-section r310c1-nrps-pks-bubbles">
      <div class="details">
    <div class="heading">
      <span>Module view</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-39"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-39">Shows module structures for each candidate in <a href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>
                     regions. Genes are shown in predicted order, and are only present when containing at least one complete module.
                     A domain glossary is available <a href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>.</span></div>
      <div class="download-container">
        <div class="download-icon download-svg" data-tag="r310c1-domain-bubble-svg" data-filename="c00310_NODE_31.._r310c1_domain_bubbles.svg">
        </div>
      </div>
    </div>
    <select id="r310c1-domain-bubble-select" class="nrps-bubble-selector">
    
      
        <option value="CC1">
          Candidate 1 (0 - 3541): single NRPS
        </option>
      
    
    </select>
    
    <div class="details-svg" id="r310c1-domain-bubble-svg-container" style="overflow:auto;">
    </div>
    <div class="bubble-legend-title">Legend</div>
    
    <div class="bubble-legend-container" id="r310c1-bubble-legend"></div>
</div>
    </div>
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
     
      <div class="sidepanel-details-header r310c1-nrps_pks_products" data-name="r310c1-nrps_pks_products">
        NRPS/PKS products
      </div>
     
      <div class="sidepanel-details-header r310c1-nrps_pks" data-name="r310c1-nrps_pks">
        NRPS/PKS monomers
      </div>
     
    
     
    
    </div>
    
     
    
     
    
     
    <div class="sidepanel-details-section r310c1-nrps_pks_products">
      <div class="structure">
  
    <div class="heading">
      <span>Predicted core structure(s)</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-37"></div> <span class="help-tooltip" id="nrps-structures-help-37">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
    </div>
    
      
        <div>
          <div class="prediction-text">
            For candidate cluster 1, location 0 - 3541:
          <div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">
          
            <canvas id="smiles-canvas-r310c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
            <div class="as-structure-warning">
              Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
            </div>
          
          
            <div>
              <dl>
                <dt><b>Polymer prediction:</b></dt><dd>(X)</dd>
                
              </dl>
              <br>Direct lookup in NORINE database:
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">strict</a>
                or
              <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
            </div>
            
            </div>
            </div>
        </div>
        
            <div id="smiles-canvas-r310c1-1-modal" class="modal-container">
              <div class="modal-content">
                <canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
              </div>
            </div>
        
      
      
    
    <div class="prediction-text">
      <br>
      <a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
    </div>
  
</div>
    </div>
     
    <div class="sidepanel-details-section r310c1-nrps_pks">
      <div class="more-details">
    <div class="heading">
      <span>NRPS/PKS monomer predictions</span>
      <div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-38"></div> <span class="help-tooltip" id="nrps-monomers-help-38">Shows the predicted monomers for each adynelation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
    </div>
  <dl class="prediction-text">
    
      <strong><span class="serif">ctg310_2</span></strong>: X
      <div class="collapser collapser-target-ctg310_2 collapser-level-cds"> </div><div class="collapser-content">
      <dt></dt>
      
        <dd> Search NORINE for peptide:
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X" target="_new">strict</a>
            or
          <a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=X*" target="_new">relaxed</a>
        </dd>
        <br>
      
      
        
          <dd><strong>AMP-binding (277..667)</strong>: X
          <div class="collapser collapser-target-ctg310_2 collapser-level-candidate"> </div><div class="collapser-content">
          
              NRPSPredictor2:  hydrophobic-aliphatic
              <div class="collapser collapser-target-ctg310_2-NRPSPredictor2 collapser-level-none"> </div><div class="collapser-content">
                
<dl><dt>SVM prediction details:</dt>
 <dd>    <dl>   <dt>Predicted physicochemical class:</dt>
   <dd>hydrophobic-aliphatic</dd>
   <dt>Large clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Small clusters prediction:</dt>
   <dd>N/A</dd>
   <dt>Single AA prediction:</dt>
   <dd>N/A</dd>
  </dl>
 </dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
 <dd>
  <dl>
   <dt>Stachelhaus sequence:</dt>
   <dd><span class="serif">DILeLaiIcK</span></dd>
   <dt>Nearest Stachelhaus code:</dt>
   <dd>gly</dd>
   <dt>Stachelhaus code match:</dt>
   <dd>60% (weak)</dd>
  </dl>
 </dd>
</dl>

              </div><br>
          
          </div>
          </dd>
        
      
        
      
      </div>
      <br>
    
  </dl>
</div>
    </div>
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r462c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00462_NODE_46.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-62"></div> <span class="help-tooltip" id="region_svg-help-62">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00462_NODE_46...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r462c1-svg" data-filename="c00462_NODE_46.._r462c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 1,902 nt. (total: 1,902 nt)
        
        <a class="cluster-rules-header" id="r462c1-rules-header" href="#r462c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r462c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r462c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-63"></div> <span class="help-tooltip" id="focus-panel-help-63">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r462c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r462c1-knownclusterblast" data-name="r462c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r462c1-subclusterblast" data-name="r462c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
    </div>
    
     
    <div class="body-details-section r462c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-40"></div> <span class="help-tooltip" id="cb-known-help-40">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r462c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-41"></div> <span class="help-tooltip" id="cb-sub-help-41">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
    </div>
    
     
    
     
    
  </div>

 </div>

</div>




<div class="page" id='r532c1' style="display: none;">
 <div class="region-grid">
  <div class = "content">
    <div class ="description-container">
      <div class="heading"> c00532_NODE_53.. - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-64"></div> <span class="help-tooltip" id="region_svg-help-64">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
      <div class="region-download">
        <a href = c00532_NODE_53...region001.gbk>Download region GenBank file</a>
      </div>
      <div class="region-download">
        <div class="link-like download-svg" data-tag="r532c1-svg" data-filename="c00532_NODE_53.._r532c1.svg">Download region SVG</div>
      </div>
      <div class = 'description-text'>
        Location: 1 - 1,409 nt. (total: 1,409 nt)
        
        <a class="cluster-rules-header" id="r532c1-rules-header" href="#r532c1">Show pHMM detection rules used</a>
        
        
          <div class="contig-edge-warning">Region on contig edge.</div>
        
      </div>
      
      <div class="cluster-rules" id="r532c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193)</div>
      
      <div class="region-svg-container">
       <div id='r532c1-svg'>
       </div>
      </div>
    </div>
    <div class="legend">
      <h4>Legend:</h4>
      <div class="legend-items">
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic">
      <div class="legend-field legend-type-biosynthetic"></div>
      <div class="legend-label">core biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
      <div class="legend-field legend-type-biosynthetic-additional"></div>
      <div class="legend-label">additional biosynthetic genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-transport">
      <div class="legend-field legend-type-transport"></div>
      <div class="legend-label">transport-related genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-regulatory">
      <div class="legend-field legend-type-regulatory"></div>
      <div class="legend-label">regulatory genes</div>
    </div>
        <div class="button-like legend-selector" data-id="legend-type-other">
      <div class="legend-field legend-type-other"></div>
      <div class="legend-label">other genes</div>
    </div>
        <div style="margin-right: 2em;"></div>
        <div class="button-like legend-selector" data-id="legend-resistance">
        <div class="legend-symbol">
            <svg viewbox="0 0 8 8">
                <rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
            </svg>
        </div>
        <div class="legend-label">resistance</div>
    </div>
        
        
      </div>
    </div>
    <div class="zoom-buttons">
      <div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
      <div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
    </div>
  </div>

  <div class="focus-panel">
    <div class="heading">
      <span>Gene details</span>
      <div class="help-container"> <div class="help-icon" data-id="focus-panel-help-65"></div> <span class="help-tooltip" id="focus-panel-help-65">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
    </div>
    <div class="focus-panel-content focus-panel-content-r532c1">
        <div style="text-align: center; margin-top: 30%;">
            Select a gene to view the details available for it
        </div>
    </div>
  </div>

  
  <div class="body-details">
    <div class="body-details-headers">
    
     
      <div class="body-details-header r532c1-knownclusterblast" data-name="r532c1-knownclusterblast">
        KnownClusterBlast
      </div>
     
      <div class="body-details-header r532c1-subclusterblast" data-name="r532c1-subclusterblast">
        SubClusterBlast
      </div>
     
    
     
    
    </div>
    
     
    <div class="body-details-section r532c1-knownclusterblast">
      <div class = "knownclusterblast">
    <div class="heading">
      <span>Similar known gene clusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-known-help-42"></div> <span class="help-tooltip" id="cb-known-help-42">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    <div class="body-details-section r532c1-subclusterblast">
      <div class = "subclusterblast">
    <div class="heading">
      <span>Similar subclusters</span>
      <div class="help-container"> <div class="help-icon" data-id="cb-sub-help-43"></div> <span class="help-tooltip" id="cb-sub-help-43">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
    </div>
  
  <div>
    <p>No matches found.</p>
  </div>
  
</div>
    </div>
     
    
     
    
  </div>

  <div class="sidepanel">
    <div class="sidepanel-details-headers">
    
     
    
     
    
    </div>
    
     
    
     
    
  </div>

 </div>

</div>


  <footer class="footer">
    <div class="container">
      <div>
        <img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
      </div>
      <div class="cite-me">
        If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
      </div>
      <div>
        <img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
      </div>
    </div>
  </footer>

  <script src="js/jquery.js"></script>
  <script src="js/antismash.js"></script>
  <script src="regions.js"></script>
  <script>
    $(document).ready(function() {
        viewer["start"](all_regions, details_data, resultsData, recordData);
    })
  </script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink">
 <defs>
  <filter id="inset-shadow">
   <feOffset dx="-2" dy="-2"></feOffset>
   <feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
   <feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
   <feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
   <feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
   <feComponentTransfer in="shadow" result="shadow">
    <feFuncA type="linear" slope=".95"></feFuncA>
   </feComponentTransfer>
   <feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
  </filter>
 </defs>
</svg>
</body>
</html>